{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AnnotationReports {}\n",
      "AssembliesReport {'target_set_accession': 'GCF_002925485.2'}\n",
      "BuildInfo {}\n",
      "FeatureCounts {}\n",
      "FeatureStats {}\n"
     ]
    }
   ],
   "source": [
    "import xml.etree.ElementTree as ET\n",
    "tree = ET.parse('Homo_sapiens_AR109.20190905_annotation_report.xml')\n",
    "root = tree.getroot()\n",
    "print(root.tag, root.attrib)\n",
    "for child in root:\n",
    "    print(child.tag, child.attrib)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4\n"
     ]
    }
   ],
   "source": [
    "# children\n",
    "count = 0 \n",
    "for child in root:\n",
    "    count += 1\n",
    "print(count)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Child:\n",
      "AssembliesReport {'target_set_accession': 'GCF_002925485.2'}\n",
      "Grandchildren:\n",
      "FullAssembly {}\n",
      "1\n",
      "\n",
      "\n",
      "\n",
      "Child:\n",
      "BuildInfo {}\n",
      "Grandchildren:\n",
      "Taxid {}\n",
      "ScientificName {}\n",
      "AnnotationReleaseId {}\n",
      "FreezeDate {}\n",
      "FreezeDateId {}\n",
      "IdLoadDate {}\n",
      "SoftwareMajorVersion {}\n",
      "SoftwareMinorVersion {}\n",
      "11\n",
      "\n",
      "\n",
      "\n",
      "Child:\n",
      "FeatureCounts {}\n",
      "Grandchildren:\n",
      "51\n",
      "\n",
      "\n",
      "\n",
      "Child:\n",
      "FeatureStats {}\n",
      "Grandchildren:\n",
      "91\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# grandchildren\n",
    "count = 0\n",
    "for child in root:\n",
    "    print('Child:')\n",
    "    print(child.tag, child.attrib)\n",
    "    print('Grandchildren:')\n",
    "    for grandchild in child:\n",
    "        count += 1\n",
    "        if count < 10:\n",
    "            print(grandchild.tag, grandchild.attrib)\n",
    "    print(count)\n",
    "    print('\\n\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Child:\n",
      "AssembliesReport {'target_set_accession': 'GCF_002925485.2'}\n",
      "Grandchildren:\n",
      "FullAssembly {}\n",
      "Great Grandchildren:  Name {}\n",
      "Great Grandchildren:  Accession {}\n",
      "Great Grandchildren:  IsReference {}\n",
      "Great Grandchildren:  SubmitterOrganization {}\n",
      "Great Grandchildren:  SubmitterDate {}\n",
      "Great Grandchildren:  AssemblyContents {}\n",
      "Great Great Grandchildren:  Chromosomes {}\n",
      "Great Great Grandchildren:  AlternateLoci {}\n",
      "203\n",
      "\n",
      "\n",
      "\n",
      "Child:\n",
      "BuildInfo {}\n",
      "Grandchildren:\n",
      "213\n",
      "\n",
      "\n",
      "\n",
      "Child:\n",
      "FeatureCounts {}\n",
      "Grandchildren:\n",
      "2621\n",
      "\n",
      "\n",
      "\n",
      "Child:\n",
      "FeatureStats {}\n",
      "Grandchildren:\n",
      "6391\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# great grandchildren\n",
    "count = 0\n",
    "for child in root:\n",
    "    print('Child:')\n",
    "    print(child.tag, child.attrib)\n",
    "    print('Grandchildren:')\n",
    "    for grandchild in child:\n",
    "        count += 1\n",
    "        if count < 10:\n",
    "            print(grandchild.tag, grandchild.attrib)\n",
    "        for greatgrandchild in grandchild:\n",
    "            count += 1\n",
    "            if count < 10:\n",
    "                print('Great Grandchildren: ', greatgrandchild.tag, greatgrandchild.attrib)\n",
    "            for greatgreatgrandchild in greatgrandchild:\n",
    "                count += 1\n",
    "                if count < 10:\n",
    "                    print('Great Great Grandchildren: ', greatgreatgrandchild.tag, greatgreatgrandchild.attrib)\n",
    "                for ggggchild in greatgreatgrandchild:\n",
    "                    count += 1\n",
    "                    if count < 10:\n",
    "                        print('Great Great Great Grandchildren: ', ggggchild.tag, ggggchild.attrib, '\\n')\n",
    "    print(count)\n",
    "    print('\\n\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6396\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['AnnotationReports',\n",
       " 'AssembliesReport',\n",
       " 'FullAssembly',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsReference',\n",
       " 'SubmitterOrganization',\n",
       " 'SubmitterDate',\n",
       " 'AssemblyContents',\n",
       " 'Chromosomes',\n",
       " 'AlternateLoci',\n",
       " 'AssemblyPatches',\n",
       " 'UnlocalizedScaffolds',\n",
       " 'UnplacedScaffolds',\n",
       " 'Units',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'AssemblyUnit',\n",
       " 'Name',\n",
       " 'Accession',\n",
       " 'IsPrimary',\n",
       " 'Type',\n",
       " 'BuildInfo',\n",
       " 'Taxid',\n",
       " 'ScientificName',\n",
       " 'AnnotationReleaseId',\n",
       " 'FreezeDate',\n",
       " 'FreezeDateId',\n",
       " 'IdLoadDate',\n",
       " 'SoftwareMajorVersion',\n",
       " 'SoftwareMinorVersion',\n",
       " 'MaskingSoftware',\n",
       " 'FtpSite',\n",
       " 'FeatureCounts',\n",
       " 'OverallCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'AssemblyCounts',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Counts',\n",
       " 'Total',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " 'Subcount',\n",
       " ...]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tags = []\n",
    "for element in root.iter():\n",
    "    tags.append(element.tag)\n",
    "print(len(tags))\n",
    "tags"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[('Name', {}, 'GRCh38.p13')],\n",
       " [('Accession', {}, 'GCF_000001405.39')],\n",
       " [('IsReference', {}, 'Yes')],\n",
       " [('SubmitterOrganization', {}, 'Genome Reference Consortium')],\n",
       " [('SubmitterDate', {}, '02-28-2019')],\n",
       " [('AssemblyContents', {}, '\\n      '),\n",
       "  ('Chromosomes', {}, '25'),\n",
       "  ('AlternateLoci', {}, None),\n",
       "  ('AssemblyPatches', {}, None),\n",
       "  ('UnlocalizedScaffolds', {}, None),\n",
       "  ('UnplacedScaffolds', {}, None)],\n",
       " [('Units', {}, '\\n      '),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'Primary Assembly'),\n",
       "  ('Accession', {}, 'GCF_000001305.15'),\n",
       "  ('IsPrimary', {}, 'Yes'),\n",
       "  ('Type', {}, 'HaploidUnit'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'PATCHES'),\n",
       "  ('Accession', {}, 'GCF_000005045.25'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AssemblyPatches'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_1'),\n",
       "  ('Accession', {}, 'GCF_000001315.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_2'),\n",
       "  ('Accession', {}, 'GCF_000001325.3'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_3'),\n",
       "  ('Accession', {}, 'GCF_000001335.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_4'),\n",
       "  ('Accession', {}, 'GCF_000001345.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_5'),\n",
       "  ('Accession', {}, 'GCF_000001355.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_6'),\n",
       "  ('Accession', {}, 'GCF_000001365.3'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_7'),\n",
       "  ('Accession', {}, 'GCF_000001375.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_8'),\n",
       "  ('Accession', {}, 'GCF_000510225.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_9'),\n",
       "  ('Accession', {}, 'GCF_000510235.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_10'),\n",
       "  ('Accession', {}, 'GCF_000509915.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_11'),\n",
       "  ('Accession', {}, 'GCF_000509925.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_12'),\n",
       "  ('Accession', {}, 'GCF_000509935.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_13'),\n",
       "  ('Accession', {}, 'GCF_000509945.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_14'),\n",
       "  ('Accession', {}, 'GCF_000509955.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_15'),\n",
       "  ('Accession', {}, 'GCF_000509965.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_16'),\n",
       "  ('Accession', {}, 'GCF_000509975.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_17'),\n",
       "  ('Accession', {}, 'GCF_000509985.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_18'),\n",
       "  ('Accession', {}, 'GCF_000509995.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_19'),\n",
       "  ('Accession', {}, 'GCF_000510005.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_20'),\n",
       "  ('Accession', {}, 'GCF_000510015.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_21'),\n",
       "  ('Accession', {}, 'GCF_000510025.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_22'),\n",
       "  ('Accession', {}, 'GCF_000510035.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_23'),\n",
       "  ('Accession', {}, 'GCF_000510045.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_24'),\n",
       "  ('Accession', {}, 'GCF_000510055.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_25'),\n",
       "  ('Accession', {}, 'GCF_000510065.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_26'),\n",
       "  ('Accession', {}, 'GCF_000510075.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_27'),\n",
       "  ('Accession', {}, 'GCF_000510085.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_28'),\n",
       "  ('Accession', {}, 'GCF_000510095.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_29'),\n",
       "  ('Accession', {}, 'GCF_000510105.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_30'),\n",
       "  ('Accession', {}, 'GCF_000510125.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_31'),\n",
       "  ('Accession', {}, 'GCF_000510135.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_32'),\n",
       "  ('Accession', {}, 'GCF_000510145.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_33'),\n",
       "  ('Accession', {}, 'GCF_000510155.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_34'),\n",
       "  ('Accession', {}, 'GCF_000510165.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_35'),\n",
       "  ('Accession', {}, 'GCF_000510175.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci'),\n",
       "  ('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'non-nuclear'),\n",
       "  ('Accession', {}, 'GCF_000006015.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'HaploidUnit')]]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_assembly_collection = []\n",
    "for i in root[0][0]:\n",
    "    tags = []\n",
    "    text = []\n",
    "    attributes = []\n",
    "    for element in i.iter():\n",
    "        tags.append(element.tag)\n",
    "        text.append(element.text)\n",
    "        attributes.append(element.attrib)\n",
    "    thing = list(zip(tags,attributes,text))\n",
    "    full_assembly_collection.append(thing)\n",
    "full_assembly_collection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[('Chromosomes', {}, '25')],\n",
       " [('AlternateLoci', {}, None)],\n",
       " [('AssemblyPatches', {}, None)],\n",
       " [('UnlocalizedScaffolds', {}, None)],\n",
       " [('UnplacedScaffolds', {}, None)]]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "assembly_contents_collection = []\n",
    "for i in root[0][0][5]:\n",
    "    tags = []\n",
    "    text = []\n",
    "    attributes = []\n",
    "    for element in i.iter():\n",
    "        tags.append(element.tag)\n",
    "        text.append(element.text)\n",
    "        attributes.append(element.attrib)\n",
    "    thing = list(zip(tags,attributes,text))\n",
    "    assembly_contents_collection.append(thing)\n",
    "assembly_contents_collection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "38\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[[('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'Primary Assembly'),\n",
       "  ('Accession', {}, 'GCF_000001305.15'),\n",
       "  ('IsPrimary', {}, 'Yes'),\n",
       "  ('Type', {}, 'HaploidUnit')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'PATCHES'),\n",
       "  ('Accession', {}, 'GCF_000005045.25'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AssemblyPatches')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_1'),\n",
       "  ('Accession', {}, 'GCF_000001315.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_2'),\n",
       "  ('Accession', {}, 'GCF_000001325.3'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_3'),\n",
       "  ('Accession', {}, 'GCF_000001335.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_4'),\n",
       "  ('Accession', {}, 'GCF_000001345.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_5'),\n",
       "  ('Accession', {}, 'GCF_000001355.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_6'),\n",
       "  ('Accession', {}, 'GCF_000001365.3'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_7'),\n",
       "  ('Accession', {}, 'GCF_000001375.2'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_8'),\n",
       "  ('Accession', {}, 'GCF_000510225.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_9'),\n",
       "  ('Accession', {}, 'GCF_000510235.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_10'),\n",
       "  ('Accession', {}, 'GCF_000509915.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_11'),\n",
       "  ('Accession', {}, 'GCF_000509925.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_12'),\n",
       "  ('Accession', {}, 'GCF_000509935.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_13'),\n",
       "  ('Accession', {}, 'GCF_000509945.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_14'),\n",
       "  ('Accession', {}, 'GCF_000509955.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_15'),\n",
       "  ('Accession', {}, 'GCF_000509965.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_16'),\n",
       "  ('Accession', {}, 'GCF_000509975.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_17'),\n",
       "  ('Accession', {}, 'GCF_000509985.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_18'),\n",
       "  ('Accession', {}, 'GCF_000509995.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_19'),\n",
       "  ('Accession', {}, 'GCF_000510005.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_20'),\n",
       "  ('Accession', {}, 'GCF_000510015.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_21'),\n",
       "  ('Accession', {}, 'GCF_000510025.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_22'),\n",
       "  ('Accession', {}, 'GCF_000510035.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_23'),\n",
       "  ('Accession', {}, 'GCF_000510045.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_24'),\n",
       "  ('Accession', {}, 'GCF_000510055.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_25'),\n",
       "  ('Accession', {}, 'GCF_000510065.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_26'),\n",
       "  ('Accession', {}, 'GCF_000510075.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_27'),\n",
       "  ('Accession', {}, 'GCF_000510085.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_28'),\n",
       "  ('Accession', {}, 'GCF_000510095.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_29'),\n",
       "  ('Accession', {}, 'GCF_000510105.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_30'),\n",
       "  ('Accession', {}, 'GCF_000510125.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_31'),\n",
       "  ('Accession', {}, 'GCF_000510135.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_32'),\n",
       "  ('Accession', {}, 'GCF_000510145.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_33'),\n",
       "  ('Accession', {}, 'GCF_000510155.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_34'),\n",
       "  ('Accession', {}, 'GCF_000510165.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'ALT_REF_LOCI_35'),\n",
       "  ('Accession', {}, 'GCF_000510175.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'AlternateLoci')],\n",
       " [('AssemblyUnit', {}, '\\n        '),\n",
       "  ('Name', {}, 'non-nuclear'),\n",
       "  ('Accession', {}, 'GCF_000006015.1'),\n",
       "  ('IsPrimary', {}, 'No'),\n",
       "  ('Type', {}, 'HaploidUnit')]]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "assembly_unit_collection = []\n",
    "for i in root[0][0][6]:\n",
    "    tags = []\n",
    "    text = []\n",
    "    attributes = []\n",
    "    for element in i.iter():\n",
    "        tags.append(element.tag)\n",
    "        text.append(element.text)\n",
    "        attributes.append(element.attrib)\n",
    "    thing = list(zip(tags, attributes, text))\n",
    "    assembly_unit_collection.append(thing)\n",
    "print(len(assembly_unit_collection))\n",
    "assembly_unit_collection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[[('Taxid', {}, '9606')],\n",
       " [('ScientificName', {}, 'Homo sapiens')],\n",
       " [('AnnotationReleaseId', {}, '109')],\n",
       " [('FreezeDate', {}, 'Sep 5 2019')],\n",
       " [('FreezeDateId', {}, '20190905')],\n",
       " [('IdLoadDate', {}, 'Sep 9 2019')],\n",
       " [('SoftwareMajorVersion', {}, '8')],\n",
       " [('SoftwareMinorVersion', {}, '2')],\n",
       " [('MaskingSoftware', {}, 'repeatmasker')],\n",
       " [('FtpSite',\n",
       "   {},\n",
       "   'ftp://ftp.ncbi.nlm.nih.gov/genomes/all/annotation_releases/9606/109.20190905')]]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "build_info_collection = []\n",
    "for i in root[1]:\n",
    "    tags = []\n",
    "    text = []\n",
    "    attributes = []\n",
    "    for element in i.iter():\n",
    "        tags.append(element.tag)\n",
    "        text.append(element.text)\n",
    "        attributes.append(element.attrib)\n",
    "    thing = list(zip(tags,attributes,text))\n",
    "    build_info_collection.append(thing)\n",
    "print(len(build_info_collection))\n",
    "build_info_collection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "40\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[[('OverallCounts', {}, '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '54639'),\n",
       "  ('Subcount', {'subtype': 'other'}, '47'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '15204'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1135'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '19965'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '364'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '397'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '18012'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '19844'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '112430'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '172'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '26'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1542'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '58361'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '54069'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1541'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '112283'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '88'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '1'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '47041'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '15'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '7'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '196'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '29863'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '15319'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '196'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '45168'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1442'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '-2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '74'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '108'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1334'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '74'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1431'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '113017'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '2600'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '514'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1109'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '58374'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '54069'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '534'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '112283'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '118'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '69'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '503'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '57'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '35')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001405.39', 'Type': 'full assembly'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '54639'),\n",
       "  ('Subcount', {'subtype': 'other'}, '47'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '15204'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1135'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '19965'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '364'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '397'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '18012'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '19844'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '3433'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '112430'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '172'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '26'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1542'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '58361'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '54069'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1541'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '112283'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '88'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '2901'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '1'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '47041'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '15'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '7'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '196'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '29863'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '15319'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '196'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '45168'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '695'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1442'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '-2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '74'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '108'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1334'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '74'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1431'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '113017'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '2600'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '514'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1109'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '58374'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '54069'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '534'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '112283'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '118'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '69'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '503'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '57'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '35')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001305.15', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '54312'),\n",
       "  ('Subcount', {'subtype': 'other'}, '45'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '15137'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1127'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '19886'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '366'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '388'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '17860'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '19755'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '112169'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '172'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '31'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1565'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '58188'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '53981'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1564'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '112038'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '78'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '1'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '45167'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '14'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '4'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '195'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '28479'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '15310'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '195'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '43775'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1426'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '78'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '97'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1329'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '78'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1417'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '112558'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '2602'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '348'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '938'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '58188'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '53981'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '548'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '112038'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '104'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '66'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '493'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '61'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '45')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001405.39', 'Type': 'grouped alt loci'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '2398'),\n",
       "  ('Subcount', {'subtype': 'other'}, '8'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '631'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '90'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '606'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '297'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '161'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '676'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '832'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '658'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '1917'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '267'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '818'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '136'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1781'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '818'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1906'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '7'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '638'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '1820'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '1'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '56'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '166'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '1013'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '506'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '170'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1519'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '183'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '109'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '9'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '65'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '8'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '101'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '65'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '107'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'multiply placed'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '2077'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '28'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '364'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '654'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '136'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1778'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '491'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1906'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '10'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '61'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '544'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '26'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '28')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001315.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '1942'),\n",
       "  ('Subcount', {'subtype': 'other'}, '8'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '515'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '71'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '486'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '305'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '161'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '529'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '658'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '1505'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '281'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '743'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '82'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1423'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '743'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1498'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '4'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '1038'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '1'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '71'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '162'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '582'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '417'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '166'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '999'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '85'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '6'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '53'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '6'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '79'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '53'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '84'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '1665'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '25'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '384'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '651'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '82'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1420'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '488'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1498'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '45'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '443'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '27'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '39')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001325.3', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '784'),\n",
       "  ('Subcount', {'subtype': 'other'}, '2'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '182'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '31'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '210'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '24'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '241'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '328'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '700'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '44'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '428'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '19'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '681'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '428'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '697'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '470'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '8'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '103'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '215'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '208'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '104'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '423'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '38'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '3'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '27'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '36'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '27'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '37'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '700'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '9'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '38'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '207'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '19'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '681'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '207'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '697'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '3'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '24'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '183'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '17'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '17')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001335.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '398'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '86'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '12'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '127'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '15'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '112'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '188'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '410'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '22'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '227'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '5'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '405'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '227'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '409'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '209'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '6'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '70'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '48'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '138'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '71'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '186'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '14'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '14'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '410'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '2'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '21'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '135'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '5'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '405'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '135'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '409'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '127'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '5'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '4')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001345.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '340'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '83'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '10'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '111'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '13'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '92'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '155'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '366'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '29'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '213'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '7'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '359'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '213'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '366'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '176'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '58'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '35'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '113'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '60'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '148'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '15'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '15'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '15'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '366'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '24'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '128'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '7'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '359'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '128'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '366'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '115'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '6'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '8')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001355.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '346'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '69'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '10'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '113'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '18'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '92'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '175'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '402'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '32'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '234'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '8'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '394'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '234'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '402'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '164'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '7'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '62'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '29'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '105'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '63'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '134'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '15'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '15'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '15'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '402'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '30'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '148'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '8'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '394'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '148'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '402'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '16'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '132'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '14'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '14')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001365.3', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '367'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '75'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '9'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '123'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '7'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '104'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '179'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '408'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '12'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '276'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '6'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '402'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '276'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '408'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '198'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '71'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '36'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '130'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '72'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '166'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '13'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '13'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '408'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '12'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '134'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '6'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '402'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '134'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '408'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '126'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '4'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '6')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000001375.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '385'),\n",
       "  ('Subcount', {'subtype': 'other'}, '1'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '82'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '13'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '124'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '16'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '112'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '177'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '420'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '37'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '265'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '5'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '415'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '265'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '420'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '206'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '73'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '46'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '125'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '74'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '171'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '419'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '2'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '28'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '171'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '5'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '414'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '171'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '420'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '160'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '7'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '7')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510225.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '45'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '5'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '24'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '3'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '8'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '31'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '92'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '3'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '52'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '92'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '52'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '92'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '21'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '7'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '14'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '21'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '92'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '40'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '92'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '40'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '92'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '39'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '1')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510235.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '65'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '9'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '34'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '11'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '43'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '126'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '73'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '4'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '122'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '73'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '126'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '32'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '14'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '18'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '32'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '6'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '6'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '6'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '126'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '49'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '4'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '122'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '49'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '126'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '48'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '1')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509915.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '15'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '5'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '11'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '19'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '19'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '19'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '19'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '19'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '19'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509925.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '11'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '10'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '10'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '10'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '10'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509935.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '8'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '0'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '14'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '14'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '14'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '14'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '14'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '14'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509945.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '11'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509955.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '14'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '5'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '10'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509965.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '15'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '3'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '2'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '13'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '13'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '3'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '3'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '13'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '13'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509975.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '8'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '0'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '13'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '13'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '13'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '13'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509985.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '11'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000509995.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '0'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510005.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '11'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510015.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '14'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '5'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '10'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510025.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '11'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510035.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '0'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510045.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '3'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '5'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '22'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '22'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '22'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '22'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '22'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '22'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '7'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '2')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510055.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '2'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '5'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '6'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '6'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '6'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '6'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '6'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '6'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510065.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '11'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '10'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '10'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '10'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '10'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510075.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '5'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '22'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '14'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '22'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '14'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '22'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '22'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '22'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '22'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '2')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510085.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '19'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '3'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '7'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '14'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '33'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '15'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '33'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '15'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '33'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '33'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '14'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '33'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '14'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '33'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '2')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510095.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '3'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '6'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '23'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '23'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '13'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '23'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '23'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '23'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '23'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '2')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510105.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '6'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '0'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '27'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '27'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '27'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '1'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '27'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '5'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '27'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '27'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '4'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '2')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510125.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '13'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '3'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '5'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '22'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '6'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '22'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '22'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '22'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '6'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '22'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '11'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '22'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '9'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '2'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '2')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510135.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '12'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '12'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '12'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '12'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '12'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '12'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '12'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510145.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '12'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '3'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '3'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '9'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '9'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '9'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '5'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510155.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '15'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '3'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '5'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '10'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '12'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510165.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '11'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '2'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '1'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '7'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '10'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '10'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '2'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '2'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '2'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '2'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '10'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '10'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '8'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000510175.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '10'),\n",
       "  ('Subcount', {'subtype': 'other'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '0'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '1'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '4'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '1'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '0'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '0'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '9'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '10'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '0'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '0'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '1'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '0'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '17'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '2'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '0'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '17'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '17'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '6'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '0')],\n",
       " [('AssemblyCounts',\n",
       "   {'Accession': 'GCF_000005045.25', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('Counts', {'feature_type': 'genes'}, '\\n      '),\n",
       "  ('Total', {}, '1529'),\n",
       "  ('Subcount', {'subtype': 'other'}, '12'),\n",
       "  ('Subcount', {'subtype': 'non-transcribed pseudo'}, '431'),\n",
       "  ('Subcount', {'subtype': 'transcribed pseudo'}, '80'),\n",
       "  ('Subcount', {'subtype': 'has variants'}, '380'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '101'),\n",
       "  ('Subcount', {'subtype': 'Ig TCR segment'}, '22'),\n",
       "  ('Subcount', {'subtype': 'non coding'}, '386'),\n",
       "  ('Subcount', {'subtype': 'protein coding'}, '598'),\n",
       "  ('Counts', {'feature_type': 'mRNAs'}, '\\n      '),\n",
       "  ('Total', {}, '1382'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '176'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '492'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '37'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1345'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '492'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1377'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '3'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Counts', {'feature_type': 'non-coding RNAs'}, '\\n      '),\n",
       "  ('Total', {}, '758'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '39'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '104'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '371'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '359'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '105'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '730'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'pseudo transcripts'}, '\\n      '),\n",
       "  ('Total', {}, '90'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '0'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '6'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '48'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '3'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '87'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '48'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '90'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '0'),\n",
       "  ('Subcount', {'subtype': 'has gaps'}, '0'),\n",
       "  ('Counts', {'feature_type': 'CDSs'}, '\\n      '),\n",
       "  ('Total', {}, '1390'),\n",
       "  ('Subcount', {'subtype': 'exon <= 3nt'}, '23'),\n",
       "  ('Subcount', {'subtype': 'partial'}, '161'),\n",
       "  ('Subcount', {'subtype': 'correction'}, '274'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq'}, '37'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq'}, '1331'),\n",
       "  ('Subcount', {'subtype': 'known RefSeq with correction'}, '252'),\n",
       "  ('Subcount', {'subtype': 'fully supported'}, '1377'),\n",
       "  ('Subcount', {'subtype': 'ab initio > 5%'}, '4'),\n",
       "  ('Subcount', {'subtype': 'model RefSeq with correction'}, '0'),\n",
       "  ('Subcount', {'subtype': 'major correction'}, '57'),\n",
       "  ('Subcount', {'subtype': 'minor correction'}, '200'),\n",
       "  ('Subcount', {'subtype': 'premature stop'}, '36'),\n",
       "  ('Subcount', {'subtype': 'has frameshifts'}, '40')]]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "feature_counts_collection = []\n",
    "for i in root[2]:\n",
    "    tags = []\n",
    "    text = []\n",
    "    attributes = []\n",
    "    for element in i.iter():\n",
    "        tags.append(element.tag)\n",
    "        text.append(element.text)\n",
    "        attributes.append(element.attrib)\n",
    "    thing = list(zip(tags, attributes ,text))\n",
    "    feature_counts_collection.append(thing)\n",
    "print(len(feature_counts_collection))\n",
    "feature_counts_collection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "40\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[[('OverallStats', {}, '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '113'),\n",
       "  ('Mean', {}, '4.181'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '363'),\n",
       "  ('Mean', {}, '11.6507'),\n",
       "  ('Median', {}, '9'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '159471'),\n",
       "  ('Min', {}, '16'),\n",
       "  ('Max', {}, '109224'),\n",
       "  ('Mean', {}, '3565.83'),\n",
       "  ('Median', {}, '2847'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '112443'),\n",
       "  ('Min', {}, '75'),\n",
       "  ('Max', {}, '107976'),\n",
       "  ('Mean', {}, '1986.65'),\n",
       "  ('Median', {}, '1461'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '37904'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '2473592'),\n",
       "  ('Mean', {}, '45183.9'),\n",
       "  ('Median', {}, '14182'),\n",
       "  ('LengthStats', {'category': 'RNase_MRP_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '277'),\n",
       "  ('Max', {}, '277'),\n",
       "  ('Mean', {}, '277'),\n",
       "  ('Median', {}, '277'),\n",
       "  ('LengthStats', {'category': 'RNase_P_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '341'),\n",
       "  ('Max', {}, '341'),\n",
       "  ('Mean', {}, '341'),\n",
       "  ('Median', {}, '341'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '1800'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2035.75'),\n",
       "  ('Median', {}, '1432'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '1403'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '12344'),\n",
       "  ('Mean', {}, '1839.05'),\n",
       "  ('Median', {}, '1198'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '397'),\n",
       "  ('Min', {}, '200'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2730.89'),\n",
       "  ('Median', {}, '2064'),\n",
       "  ('LengthStats', {'category': 'Y_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '113'),\n",
       "  ('Mean', {}, '98.75'),\n",
       "  ('Median', {}, '102'),\n",
       "  ('LengthStats', {'category': 'antisense_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '255'),\n",
       "  ('Max', {}, '37027'),\n",
       "  ('Mean', {}, '4169.59'),\n",
       "  ('Median', {}, '1662'),\n",
       "  ('LengthStats', {'category': 'guide_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '29'),\n",
       "  ('Min', {}, '125'),\n",
       "  ('Max', {}, '421'),\n",
       "  ('Mean', {}, '206.793'),\n",
       "  ('Median', {}, '148'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '27344'),\n",
       "  ('Min', {}, '99'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2029.63'),\n",
       "  ('Median', {}, '1376'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '112430'),\n",
       "  ('Min', {}, '153'),\n",
       "  ('Max', {}, '109224'),\n",
       "  ('Mean', {}, '4072.68'),\n",
       "  ('Median', {}, '3313'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '3173'),\n",
       "  ('Min', {}, '16'),\n",
       "  ('Max', {}, '28'),\n",
       "  ('Mean', {}, '21.6306'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '15218'),\n",
       "  ('Min', {}, '158'),\n",
       "  ('Max', {}, '25390'),\n",
       "  ('Mean', {}, '3605.73'),\n",
       "  ('Median', {}, '2981'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '1913'),\n",
       "  ('Min', {}, '41'),\n",
       "  ('Max', {}, '180'),\n",
       "  ('Mean', {}, '81.9226'),\n",
       "  ('Median', {}, '80'),\n",
       "  ('LengthStats', {'category': 'rRNA'}, '\\n      '),\n",
       "  ('Count', {}, '39'),\n",
       "  ('Min', {}, '121'),\n",
       "  ('Max', {}, '13373'),\n",
       "  ('Mean', {}, '2736.56'),\n",
       "  ('Median', {}, '157'),\n",
       "  ('LengthStats', {'category': 'scRNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '58'),\n",
       "  ('Max', {}, '299'),\n",
       "  ('Mean', {}, '238.75'),\n",
       "  ('Median', {}, '299'),\n",
       "  ('LengthStats', {'category': 'snRNA'}, '\\n      '),\n",
       "  ('Count', {}, '63'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '332'),\n",
       "  ('Mean', {}, '134.921'),\n",
       "  ('Median', {}, '121'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '541'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '250'),\n",
       "  ('Mean', {}, '105.649'),\n",
       "  ('Median', {}, '94'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '597'),\n",
       "  ('Min', {}, '59'),\n",
       "  ('Max', {}, '84'),\n",
       "  ('Mean', {}, '73.7672'),\n",
       "  ('Median', {}, '73'),\n",
       "  ('LengthStats', {'category': 'telomerase_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '451'),\n",
       "  ('Max', {}, '451'),\n",
       "  ('Mean', {}, '451'),\n",
       "  ('Median', {}, '451'),\n",
       "  ('LengthStats', {'category': 'vault_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '88'),\n",
       "  ('Max', {}, '108'),\n",
       "  ('Mean', {}, '95.5'),\n",
       "  ('Median', {}, '98')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001405.39', 'Type': 'full assembly'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '113'),\n",
       "  ('Mean', {}, '4.181'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '363'),\n",
       "  ('Mean', {}, '11.6507'),\n",
       "  ('Median', {}, '9'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '411273'),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '441.791'),\n",
       "  ('Median', {}, '149'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '352207'),\n",
       "  ('Min', {}, '26'),\n",
       "  ('Max', {}, '1160411'),\n",
       "  ('Mean', {}, '8185.69'),\n",
       "  ('Median', {}, '1873'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '159471'),\n",
       "  ('Min', {}, '16'),\n",
       "  ('Max', {}, '109224'),\n",
       "  ('Mean', {}, '3565.83'),\n",
       "  ('Median', {}, '2847'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '112443'),\n",
       "  ('Min', {}, '75'),\n",
       "  ('Max', {}, '107976'),\n",
       "  ('Mean', {}, '1986.65'),\n",
       "  ('Median', {}, '1461'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '327126'),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '33290'),\n",
       "  ('Mean', {}, '413.104'),\n",
       "  ('Median', {}, '143'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '156519'),\n",
       "  ('Min', {}, '2'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '417.327'),\n",
       "  ('Median', {}, '147'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '37904'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '2473592'),\n",
       "  ('Mean', {}, '45183.9'),\n",
       "  ('Median', {}, '14182'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '288015'),\n",
       "  ('Min', {}, '26'),\n",
       "  ('Max', {}, '1160411'),\n",
       "  ('Mean', {}, '7752.87'),\n",
       "  ('Median', {}, '1759'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '132792'),\n",
       "  ('Min', {}, '28'),\n",
       "  ('Max', {}, '773192'),\n",
       "  ('Mean', {}, '7841.45'),\n",
       "  ('Median', {}, '1965'),\n",
       "  ('LengthStats', {'category': 'RNase_MRP_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '277'),\n",
       "  ('Max', {}, '277'),\n",
       "  ('Mean', {}, '277'),\n",
       "  ('Median', {}, '277'),\n",
       "  ('LengthStats', {'category': 'RNase_P_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '341'),\n",
       "  ('Max', {}, '341'),\n",
       "  ('Mean', {}, '341'),\n",
       "  ('Median', {}, '341'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '1800'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2035.75'),\n",
       "  ('Median', {}, '1432'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '1403'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '12344'),\n",
       "  ('Mean', {}, '1839.05'),\n",
       "  ('Median', {}, '1198'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '397'),\n",
       "  ('Min', {}, '200'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2730.89'),\n",
       "  ('Median', {}, '2064'),\n",
       "  ('LengthStats', {'category': 'Y_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '113'),\n",
       "  ('Mean', {}, '98.75'),\n",
       "  ('Median', {}, '102'),\n",
       "  ('LengthStats', {'category': 'antisense_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '255'),\n",
       "  ('Max', {}, '37027'),\n",
       "  ('Mean', {}, '4169.59'),\n",
       "  ('Median', {}, '1662'),\n",
       "  ('LengthStats', {'category': 'guide_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '29'),\n",
       "  ('Min', {}, '125'),\n",
       "  ('Max', {}, '421'),\n",
       "  ('Mean', {}, '206.793'),\n",
       "  ('Median', {}, '148'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '27344'),\n",
       "  ('Min', {}, '99'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2029.63'),\n",
       "  ('Median', {}, '1376'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '112430'),\n",
       "  ('Min', {}, '153'),\n",
       "  ('Max', {}, '109224'),\n",
       "  ('Mean', {}, '4072.68'),\n",
       "  ('Median', {}, '3313'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '3173'),\n",
       "  ('Min', {}, '16'),\n",
       "  ('Max', {}, '28'),\n",
       "  ('Mean', {}, '21.6306'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '15218'),\n",
       "  ('Min', {}, '158'),\n",
       "  ('Max', {}, '25390'),\n",
       "  ('Mean', {}, '3605.73'),\n",
       "  ('Median', {}, '2981'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '1913'),\n",
       "  ('Min', {}, '41'),\n",
       "  ('Max', {}, '180'),\n",
       "  ('Mean', {}, '81.9226'),\n",
       "  ('Median', {}, '80'),\n",
       "  ('LengthStats', {'category': 'rRNA'}, '\\n      '),\n",
       "  ('Count', {}, '39'),\n",
       "  ('Min', {}, '121'),\n",
       "  ('Max', {}, '13373'),\n",
       "  ('Mean', {}, '2736.56'),\n",
       "  ('Median', {}, '157'),\n",
       "  ('LengthStats', {'category': 'scRNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '58'),\n",
       "  ('Max', {}, '299'),\n",
       "  ('Mean', {}, '238.75'),\n",
       "  ('Median', {}, '299'),\n",
       "  ('LengthStats', {'category': 'snRNA'}, '\\n      '),\n",
       "  ('Count', {}, '63'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '332'),\n",
       "  ('Mean', {}, '134.921'),\n",
       "  ('Median', {}, '121'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '541'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '250'),\n",
       "  ('Mean', {}, '105.649'),\n",
       "  ('Median', {}, '94'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '597'),\n",
       "  ('Min', {}, '59'),\n",
       "  ('Max', {}, '84'),\n",
       "  ('Mean', {}, '73.7672'),\n",
       "  ('Median', {}, '73'),\n",
       "  ('LengthStats', {'category': 'telomerase_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '451'),\n",
       "  ('Max', {}, '451'),\n",
       "  ('Mean', {}, '451'),\n",
       "  ('Median', {}, '451'),\n",
       "  ('LengthStats', {'category': 'vault_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '88'),\n",
       "  ('Max', {}, '108'),\n",
       "  ('Mean', {}, '95.5'),\n",
       "  ('Median', {}, '98')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001305.15', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '113'),\n",
       "  ('Mean', {}, '4.18929'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '363'),\n",
       "  ('Mean', {}, '11.7428'),\n",
       "  ('Median', {}, '9'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '378131'),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '451.678'),\n",
       "  ('Median', {}, '149'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '323298'),\n",
       "  ('Min', {}, '26'),\n",
       "  ('Max', {}, '1160411'),\n",
       "  ('Mean', {}, '8626.91'),\n",
       "  ('Median', {}, '2004'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '157336'),\n",
       "  ('Min', {}, '16'),\n",
       "  ('Max', {}, '109224'),\n",
       "  ('Mean', {}, '3594.62'),\n",
       "  ('Median', {}, '2873'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '112169'),\n",
       "  ('Min', {}, '75'),\n",
       "  ('Max', {}, '107976'),\n",
       "  ('Mean', {}, '1988.61'),\n",
       "  ('Median', {}, '1461'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '299699'),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '33290'),\n",
       "  ('Mean', {}, '424.016'),\n",
       "  ('Median', {}, '144'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '147954'),\n",
       "  ('Min', {}, '2'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '418.11'),\n",
       "  ('Median', {}, '146'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '37660'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '2473592'),\n",
       "  ('Mean', {}, '45795.7'),\n",
       "  ('Median', {}, '14350'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '263421'),\n",
       "  ('Min', {}, '26'),\n",
       "  ('Max', {}, '1160411'),\n",
       "  ('Mean', {}, '8220.54'),\n",
       "  ('Median', {}, '1904'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '125972'),\n",
       "  ('Min', {}, '28'),\n",
       "  ('Max', {}, '773192'),\n",
       "  ('Mean', {}, '8006.62'),\n",
       "  ('Median', {}, '2015'),\n",
       "  ('LengthStats', {'category': 'RNase_MRP_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '277'),\n",
       "  ('Max', {}, '277'),\n",
       "  ('Mean', {}, '277'),\n",
       "  ('Median', {}, '277'),\n",
       "  ('LengthStats', {'category': 'RNase_P_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '341'),\n",
       "  ('Max', {}, '341'),\n",
       "  ('Mean', {}, '341'),\n",
       "  ('Median', {}, '341'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '1769'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2041.42'),\n",
       "  ('Median', {}, '1434'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '1376'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '12344'),\n",
       "  ('Mean', {}, '1844.77'),\n",
       "  ('Median', {}, '1201'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '393'),\n",
       "  ('Min', {}, '200'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2729.97'),\n",
       "  ('Median', {}, '2064'),\n",
       "  ('LengthStats', {'category': 'Y_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '113'),\n",
       "  ('Mean', {}, '98.75'),\n",
       "  ('Median', {}, '102'),\n",
       "  ('LengthStats', {'category': 'antisense_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '255'),\n",
       "  ('Max', {}, '37027'),\n",
       "  ('Mean', {}, '4169.59'),\n",
       "  ('Median', {}, '1662'),\n",
       "  ('LengthStats', {'category': 'guide_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '29'),\n",
       "  ('Min', {}, '125'),\n",
       "  ('Max', {}, '421'),\n",
       "  ('Mean', {}, '206.793'),\n",
       "  ('Median', {}, '148'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '26085'),\n",
       "  ('Min', {}, '99'),\n",
       "  ('Max', {}, '91671'),\n",
       "  ('Mean', {}, '2044.77'),\n",
       "  ('Median', {}, '1388'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '112169'),\n",
       "  ('Min', {}, '153'),\n",
       "  ('Max', {}, '109224'),\n",
       "  ('Mean', {}, '4077.3'),\n",
       "  ('Median', {}, '3319'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2875'),\n",
       "  ('Min', {}, '16'),\n",
       "  ('Max', {}, '28'),\n",
       "  ('Mean', {}, '21.6303'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '15084'),\n",
       "  ('Min', {}, '158'),\n",
       "  ('Max', {}, '25390'),\n",
       "  ('Mean', {}, '3614.11'),\n",
       "  ('Median', {}, '2987'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '1913'),\n",
       "  ('Min', {}, '41'),\n",
       "  ('Max', {}, '180'),\n",
       "  ('Mean', {}, '81.9226'),\n",
       "  ('Median', {}, '80'),\n",
       "  ('LengthStats', {'category': 'rRNA'}, '\\n      '),\n",
       "  ('Count', {}, '37'),\n",
       "  ('Min', {}, '121'),\n",
       "  ('Max', {}, '13373'),\n",
       "  ('Mean', {}, '2816.57'),\n",
       "  ('Median', {}, '157'),\n",
       "  ('LengthStats', {'category': 'scRNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '58'),\n",
       "  ('Max', {}, '299'),\n",
       "  ('Mean', {}, '238.75'),\n",
       "  ('Median', {}, '299'),\n",
       "  ('LengthStats', {'category': 'snRNA'}, '\\n      '),\n",
       "  ('Count', {}, '63'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '332'),\n",
       "  ('Mean', {}, '134.921'),\n",
       "  ('Median', {}, '121'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '541'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '250'),\n",
       "  ('Mean', {}, '105.649'),\n",
       "  ('Median', {}, '94'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '416'),\n",
       "  ('Min', {}, '71'),\n",
       "  ('Max', {}, '84'),\n",
       "  ('Mean', {}, '73.9423'),\n",
       "  ('Median', {}, '73'),\n",
       "  ('LengthStats', {'category': 'telomerase_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '451'),\n",
       "  ('Max', {}, '451'),\n",
       "  ('Mean', {}, '451'),\n",
       "  ('Median', {}, '451'),\n",
       "  ('LengthStats', {'category': 'vault_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '88'),\n",
       "  ('Max', {}, '108'),\n",
       "  ('Mean', {}, '95.5'),\n",
       "  ('Median', {}, '98')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001405.39', 'Type': 'grouped alt loci'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '36'),\n",
       "  ('Mean', {}, '2.14198'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '66'),\n",
       "  ('Mean', {}, '7.05033'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '24956'),\n",
       "  ('Min', {}, '2'),\n",
       "  ('Max', {}, '18726'),\n",
       "  ('Mean', {}, '316.512'),\n",
       "  ('Median', {}, '143'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '21748'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '232263'),\n",
       "  ('Mean', {}, '2901.07'),\n",
       "  ('Median', {}, '742'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '3737'),\n",
       "  ('Min', {}, '17'),\n",
       "  ('Max', {}, '22824'),\n",
       "  ('Mean', {}, '2135.54'),\n",
       "  ('Median', {}, '1649'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '1917'),\n",
       "  ('Min', {}, '75'),\n",
       "  ('Max', {}, '22257'),\n",
       "  ('Mean', {}, '1561.41'),\n",
       "  ('Median', {}, '1182'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '20697'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '18726'),\n",
       "  ('Mean', {}, '278.942'),\n",
       "  ('Median', {}, '135'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '6523'),\n",
       "  ('Min', {}, '2'),\n",
       "  ('Max', {}, '9896'),\n",
       "  ('Mean', {}, '396.419'),\n",
       "  ('Median', {}, '162'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '1517'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '1296601'),\n",
       "  ('Mean', {}, '22661.2'),\n",
       "  ('Median', {}, '7270'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '18560'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '197352'),\n",
       "  ('Mean', {}, '2376.67'),\n",
       "  ('Median', {}, '647'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '5178'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '232263'),\n",
       "  ('Mean', {}, '4504.26'),\n",
       "  ('Median', {}, '983'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '159'),\n",
       "  ('Min', {}, '252'),\n",
       "  ('Max', {}, '8155'),\n",
       "  ('Mean', {}, '1895.09'),\n",
       "  ('Median', {}, '1100'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '127'),\n",
       "  ('Min', {}, '445'),\n",
       "  ('Max', {}, '7528'),\n",
       "  ('Mean', {}, '1752.8'),\n",
       "  ('Median', {}, '1041'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '32'),\n",
       "  ('Min', {}, '252'),\n",
       "  ('Max', {}, '8155'),\n",
       "  ('Mean', {}, '2459.81'),\n",
       "  ('Median', {}, '2175'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '1204'),\n",
       "  ('Min', {}, '99'),\n",
       "  ('Max', {}, '11661'),\n",
       "  ('Mean', {}, '1654.36'),\n",
       "  ('Median', {}, '1162'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '1917'),\n",
       "  ('Min', {}, '264'),\n",
       "  ('Max', {}, '22824'),\n",
       "  ('Mean', {}, '2828.26'),\n",
       "  ('Median', {}, '2252'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '215'),\n",
       "  ('Min', {}, '17'),\n",
       "  ('Max', {}, '25'),\n",
       "  ('Mean', {}, '21.6326'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '232'),\n",
       "  ('Min', {}, '207'),\n",
       "  ('Max', {}, '6314'),\n",
       "  ('Mean', {}, '2368.12'),\n",
       "  ('Median', {}, '2216'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '71'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '153'),\n",
       "  ('Mean', {}, '80.7324'),\n",
       "  ('Median', {}, '77'),\n",
       "  ('LengthStats', {'category': 'snRNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '165'),\n",
       "  ('Max', {}, '165'),\n",
       "  ('Mean', {}, '165'),\n",
       "  ('Median', {}, '165'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '175'),\n",
       "  ('Mean', {}, '96.9091'),\n",
       "  ('Median', {}, '78'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '157'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '83'),\n",
       "  ('Mean', {}, '73.9299'),\n",
       "  ('Median', {}, '73')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001315.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '21'),\n",
       "  ('Mean', {}, '2.13567'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '60'),\n",
       "  ('Mean', {}, '7.20447'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '8112'),\n",
       "  ('Min', {}, '2'),\n",
       "  ('Max', {}, '18726'),\n",
       "  ('Mean', {}, '360.229'),\n",
       "  ('Median', {}, '146'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '6903'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '232263'),\n",
       "  ('Mean', {}, '4463.71'),\n",
       "  ('Median', {}, '1276'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '2543'),\n",
       "  ('Min', {}, '17'),\n",
       "  ('Max', {}, '22824'),\n",
       "  ('Mean', {}, '2349.12'),\n",
       "  ('Median', {}, '1890'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '1505'),\n",
       "  ('Min', {}, '75'),\n",
       "  ('Max', {}, '22257'),\n",
       "  ('Mean', {}, '1550.77'),\n",
       "  ('Median', {}, '1233'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '6114'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '18726'),\n",
       "  ('Mean', {}, '308.109'),\n",
       "  ('Median', {}, '137'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '2695'),\n",
       "  ('Min', {}, '2'),\n",
       "  ('Max', {}, '9896'),\n",
       "  ('Mean', {}, '441.574'),\n",
       "  ('Median', {}, '171'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '1195'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '1296601'),\n",
       "  ('Mean', {}, '23813.6'),\n",
       "  ('Median', {}, '7853'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '5448'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '176607'),\n",
       "  ('Mean', {}, '4070.42'),\n",
       "  ('Median', {}, '1211'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '2067'),\n",
       "  ('Min', {}, '54'),\n",
       "  ('Max', {}, '232263'),\n",
       "  ('Mean', {}, '5248.83'),\n",
       "  ('Median', {}, '1413'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '143'),\n",
       "  ('Min', {}, '252'),\n",
       "  ('Max', {}, '8155'),\n",
       "  ('Mean', {}, '1922.1'),\n",
       "  ('Median', {}, '1096'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '113'),\n",
       "  ('Min', {}, '445'),\n",
       "  ('Max', {}, '7528'),\n",
       "  ('Mean', {}, '1757.2'),\n",
       "  ('Median', {}, '1039'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '30'),\n",
       "  ('Min', {}, '252'),\n",
       "  ('Max', {}, '8155'),\n",
       "  ('Mean', {}, '2543.2'),\n",
       "  ('Median', {}, '2435'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '763'),\n",
       "  ('Min', {}, '128'),\n",
       "  ('Max', {}, '11661'),\n",
       "  ('Mean', {}, '1726.92'),\n",
       "  ('Median', {}, '1230'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '1505'),\n",
       "  ('Min', {}, '264'),\n",
       "  ('Max', {}, '22824'),\n",
       "  ('Mean', {}, '2853.48'),\n",
       "  ('Median', {}, '2304'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '80'),\n",
       "  ('Min', {}, '17'),\n",
       "  ('Max', {}, '25'),\n",
       "  ('Mean', {}, '21.7375'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '167'),\n",
       "  ('Min', {}, '207'),\n",
       "  ('Max', {}, '6314'),\n",
       "  ('Mean', {}, '2141.28'),\n",
       "  ('Median', {}, '2006'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '59'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '153'),\n",
       "  ('Mean', {}, '79.4407'),\n",
       "  ('Median', {}, '77'),\n",
       "  ('LengthStats', {'category': 'snRNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '165'),\n",
       "  ('Max', {}, '165'),\n",
       "  ('Mean', {}, '165'),\n",
       "  ('Median', {}, '165'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '175'),\n",
       "  ('Mean', {}, '95.1111'),\n",
       "  ('Median', {}, '77'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '18'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '83'),\n",
       "  ('Mean', {}, '73.7222'),\n",
       "  ('Median', {}, '73')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001325.3', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '36'),\n",
       "  ('Mean', {}, '2.10694'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '66'),\n",
       "  ('Mean', {}, '7.41941'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '3876'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '14433'),\n",
       "  ('Mean', {}, '355.506'),\n",
       "  ('Median', {}, '142'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '3336'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '197352'),\n",
       "  ('Mean', {}, '3610.37'),\n",
       "  ('Median', {}, '656'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '1170'),\n",
       "  ('Min', {}, '18'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2275.08'),\n",
       "  ('Median', {}, '1870'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '700'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '16239'),\n",
       "  ('Mean', {}, '1541.12'),\n",
       "  ('Median', {}, '1044'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3119'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '14433'),\n",
       "  ('Mean', {}, '305.422'),\n",
       "  ('Median', {}, '134'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1087'),\n",
       "  ('Min', {}, '12'),\n",
       "  ('Max', {}, '7568'),\n",
       "  ('Mean', {}, '456.215'),\n",
       "  ('Median', {}, '165'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '571'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '460024'),\n",
       "  ('Mean', {}, '19216.8'),\n",
       "  ('Median', {}, '5361'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '2766'),\n",
       "  ('Min', {}, '33'),\n",
       "  ('Max', {}, '197352'),\n",
       "  ('Mean', {}, '2895.93'),\n",
       "  ('Median', {}, '565'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '862'),\n",
       "  ('Min', {}, '71'),\n",
       "  ('Max', {}, '124245'),\n",
       "  ('Mean', {}, '5756.43'),\n",
       "  ('Median', {}, '1182'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '65'),\n",
       "  ('Min', {}, '445'),\n",
       "  ('Max', {}, '4834'),\n",
       "  ('Mean', {}, '1556.38'),\n",
       "  ('Median', {}, '1075'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '56'),\n",
       "  ('Min', {}, '445'),\n",
       "  ('Max', {}, '4834'),\n",
       "  ('Mean', {}, '1451.48'),\n",
       "  ('Median', {}, '1048'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '770'),\n",
       "  ('Max', {}, '4805'),\n",
       "  ('Mean', {}, '2209.11'),\n",
       "  ('Median', {}, '2083'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '312'),\n",
       "  ('Min', {}, '99'),\n",
       "  ('Max', {}, '9818'),\n",
       "  ('Mean', {}, '1756.58'),\n",
       "  ('Median', {}, '1219'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '700'),\n",
       "  ('Min', {}, '331'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2663.46'),\n",
       "  ('Median', {}, '2131'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '30'),\n",
       "  ('Min', {}, '18'),\n",
       "  ('Max', {}, '23'),\n",
       "  ('Mean', {}, '21.4667'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '86'),\n",
       "  ('Min', {}, '310'),\n",
       "  ('Max', {}, '6314'),\n",
       "  ('Mean', {}, '2855.48'),\n",
       "  ('Median', {}, '2746'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '19'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '110'),\n",
       "  ('Mean', {}, '83.7895'),\n",
       "  ('Median', {}, '86'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '132'),\n",
       "  ('Mean', {}, '81.6667'),\n",
       "  ('Median', {}, '77'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '36'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '83'),\n",
       "  ('Mean', {}, '73.9444'),\n",
       "  ('Median', {}, '73')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001335.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '13'),\n",
       "  ('Mean', {}, '2.08392'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '66'),\n",
       "  ('Mean', {}, '8.05705'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '2281'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '8265'),\n",
       "  ('Mean', {}, '301.163'),\n",
       "  ('Median', {}, '143'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '2003'),\n",
       "  ('Min', {}, '48'),\n",
       "  ('Max', {}, '110063'),\n",
       "  ('Mean', {}, '1855.19'),\n",
       "  ('Median', {}, '433'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '619'),\n",
       "  ('Min', {}, '20'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2027.55'),\n",
       "  ('Median', {}, '1675'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '410'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '16239'),\n",
       "  ('Mean', {}, '1595.51'),\n",
       "  ('Median', {}, '1071'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1961'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '8265'),\n",
       "  ('Mean', {}, '273.348'),\n",
       "  ('Median', {}, '138'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '519'),\n",
       "  ('Min', {}, '11'),\n",
       "  ('Max', {}, '7568'),\n",
       "  ('Mean', {}, '366.651'),\n",
       "  ('Median', {}, '154'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '300'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '152246'),\n",
       "  ('Mean', {}, '11334.9'),\n",
       "  ('Median', {}, '5586'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1751'),\n",
       "  ('Min', {}, '48'),\n",
       "  ('Max', {}, '35672'),\n",
       "  ('Mean', {}, '1161.36'),\n",
       "  ('Median', {}, '366'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '424'),\n",
       "  ('Min', {}, '71'),\n",
       "  ('Max', {}, '110063'),\n",
       "  ('Mean', {}, '4483.21'),\n",
       "  ('Median', {}, '964'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '24'),\n",
       "  ('Min', {}, '963'),\n",
       "  ('Max', {}, '2624'),\n",
       "  ('Mean', {}, '1634.71'),\n",
       "  ('Median', {}, '1348'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '18'),\n",
       "  ('Min', {}, '963'),\n",
       "  ('Max', {}, '2517'),\n",
       "  ('Mean', {}, '1531.33'),\n",
       "  ('Median', {}, '1260'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '1180'),\n",
       "  ('Max', {}, '2624'),\n",
       "  ('Mean', {}, '1944.83'),\n",
       "  ('Median', {}, '2083'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '139'),\n",
       "  ('Min', {}, '149'),\n",
       "  ('Max', {}, '8477'),\n",
       "  ('Mean', {}, '1434.51'),\n",
       "  ('Median', {}, '893'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '410'),\n",
       "  ('Min', {}, '402'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2428.95'),\n",
       "  ('Median', {}, '2059'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '20'),\n",
       "  ('Min', {}, '20'),\n",
       "  ('Max', {}, '23'),\n",
       "  ('Mean', {}, '21.65'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '30'),\n",
       "  ('Min', {}, '505'),\n",
       "  ('Max', {}, '4543'),\n",
       "  ('Mean', {}, '1928.1'),\n",
       "  ('Median', {}, '1801'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '110'),\n",
       "  ('Mean', {}, '81.8182'),\n",
       "  ('Median', {}, '81'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '132'),\n",
       "  ('Mean', {}, '81.6667'),\n",
       "  ('Median', {}, '77'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '82'),\n",
       "  ('Mean', {}, '73.0714'),\n",
       "  ('Median', {}, '72')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001345.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '13'),\n",
       "  ('Mean', {}, '2.26432'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '66'),\n",
       "  ('Mean', {}, '7.55642'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '1708'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '12739'),\n",
       "  ('Mean', {}, '306.721'),\n",
       "  ('Median', {}, '149'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '1492'),\n",
       "  ('Min', {}, '68'),\n",
       "  ('Max', {}, '110356'),\n",
       "  ('Mean', {}, '1996.42'),\n",
       "  ('Median', {}, '535'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '542'),\n",
       "  ('Min', {}, '19'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '1938.63'),\n",
       "  ('Median', {}, '1603'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '366'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '16239'),\n",
       "  ('Mean', {}, '1458.86'),\n",
       "  ('Median', {}, '1002'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1463'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '12739'),\n",
       "  ('Mean', {}, '279.636'),\n",
       "  ('Median', {}, '142'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '432'),\n",
       "  ('Min', {}, '23'),\n",
       "  ('Max', {}, '7568'),\n",
       "  ('Mean', {}, '343.002'),\n",
       "  ('Median', {}, '153'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '247'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '152594'),\n",
       "  ('Mean', {}, '10004.8'),\n",
       "  ('Median', {}, '5355'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1293'),\n",
       "  ('Min', {}, '68'),\n",
       "  ('Max', {}, '53017'),\n",
       "  ('Mean', {}, '1237.51'),\n",
       "  ('Median', {}, '453'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '361'),\n",
       "  ('Min', {}, '71'),\n",
       "  ('Max', {}, '110356'),\n",
       "  ('Mean', {}, '4303.32'),\n",
       "  ('Median', {}, '861'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '20'),\n",
       "  ('Min', {}, '770'),\n",
       "  ('Max', {}, '2624'),\n",
       "  ('Mean', {}, '1417.75'),\n",
       "  ('Median', {}, '1136'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '15'),\n",
       "  ('Min', {}, '963'),\n",
       "  ('Max', {}, '2481'),\n",
       "  ('Mean', {}, '1310.47'),\n",
       "  ('Median', {}, '1075'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '5'),\n",
       "  ('Min', {}, '770'),\n",
       "  ('Max', {}, '2624'),\n",
       "  ('Mean', {}, '1739.6'),\n",
       "  ('Median', {}, '1889'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '108'),\n",
       "  ('Min', {}, '99'),\n",
       "  ('Max', {}, '8477'),\n",
       "  ('Mean', {}, '1469.68'),\n",
       "  ('Median', {}, '955'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '366'),\n",
       "  ('Min', {}, '402'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2290.78'),\n",
       "  ('Median', {}, '1970'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '18'),\n",
       "  ('Min', {}, '19'),\n",
       "  ('Max', {}, '23'),\n",
       "  ('Mean', {}, '21.5'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '26'),\n",
       "  ('Min', {}, '505'),\n",
       "  ('Max', {}, '6314'),\n",
       "  ('Mean', {}, '1975.19'),\n",
       "  ('Median', {}, '1571'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '110'),\n",
       "  ('Mean', {}, '82.7'),\n",
       "  ('Median', {}, '87'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '76'),\n",
       "  ('Max', {}, '132'),\n",
       "  ('Mean', {}, '90.75'),\n",
       "  ('Median', {}, '78'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '20'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '83'),\n",
       "  ('Mean', {}, '73.95'),\n",
       "  ('Median', {}, '73')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001355.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '13'),\n",
       "  ('Mean', {}, '2.17004'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '66'),\n",
       "  ('Mean', {}, '8.30224'),\n",
       "  ('Median', {}, '6'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '2015'),\n",
       "  ('Min', {}, '11'),\n",
       "  ('Max', {}, '7195'),\n",
       "  ('Mean', {}, '277.085'),\n",
       "  ('Median', {}, '141'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '1759'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '39615'),\n",
       "  ('Mean', {}, '1315.9'),\n",
       "  ('Median', {}, '398'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '566'),\n",
       "  ('Min', {}, '20'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '1938.92'),\n",
       "  ('Median', {}, '1632'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '402'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '16239'),\n",
       "  ('Mean', {}, '1513.62'),\n",
       "  ('Median', {}, '1044'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1800'),\n",
       "  ('Min', {}, '12'),\n",
       "  ('Max', {}, '7195'),\n",
       "  ('Mean', {}, '261.909'),\n",
       "  ('Median', {}, '138'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '432'),\n",
       "  ('Min', {}, '11'),\n",
       "  ('Max', {}, '3210'),\n",
       "  ('Mean', {}, '295.051'),\n",
       "  ('Median', {}, '143'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '267'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '152244'),\n",
       "  ('Mean', {}, '9200.34'),\n",
       "  ('Median', {}, '5056'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1605'),\n",
       "  ('Min', {}, '48'),\n",
       "  ('Max', {}, '36388'),\n",
       "  ('Mean', {}, '1140.18'),\n",
       "  ('Median', {}, '352'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '347'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '39615'),\n",
       "  ('Mean', {}, '2001.61'),\n",
       "  ('Median', {}, '594'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '24'),\n",
       "  ('Min', {}, '407'),\n",
       "  ('Max', {}, '2547'),\n",
       "  ('Mean', {}, '1351'),\n",
       "  ('Median', {}, '1136'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '16'),\n",
       "  ('Min', {}, '963'),\n",
       "  ('Max', {}, '2481'),\n",
       "  ('Mean', {}, '1335'),\n",
       "  ('Median', {}, '1106'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '407'),\n",
       "  ('Max', {}, '2547'),\n",
       "  ('Mean', {}, '1383'),\n",
       "  ('Median', {}, '1332'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '89'),\n",
       "  ('Min', {}, '333'),\n",
       "  ('Max', {}, '4070'),\n",
       "  ('Mean', {}, '1238.73'),\n",
       "  ('Median', {}, '940'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '402'),\n",
       "  ('Min', {}, '402'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2303.87'),\n",
       "  ('Median', {}, '2005'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '20'),\n",
       "  ('Max', {}, '23'),\n",
       "  ('Mean', {}, '21.6818'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '29'),\n",
       "  ('Min', {}, '505'),\n",
       "  ('Max', {}, '6314'),\n",
       "  ('Mean', {}, '2024.31'),\n",
       "  ('Median', {}, '1603'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '110'),\n",
       "  ('Mean', {}, '81.6667'),\n",
       "  ('Median', {}, '86'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '76'),\n",
       "  ('Max', {}, '132'),\n",
       "  ('Mean', {}, '90.75'),\n",
       "  ('Median', {}, '78'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '20'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '83'),\n",
       "  ('Mean', {}, '74.1'),\n",
       "  ('Median', {}, '73')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001365.3', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '13'),\n",
       "  ('Mean', {}, '2.19923'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '66'),\n",
       "  ('Mean', {}, '8.15505'),\n",
       "  ('Median', {}, '6'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '2083'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '12739'),\n",
       "  ('Mean', {}, '293.904'),\n",
       "  ('Median', {}, '144'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '1832'),\n",
       "  ('Min', {}, '45'),\n",
       "  ('Max', {}, '110042'),\n",
       "  ('Mean', {}, '1644.74'),\n",
       "  ('Median', {}, '437'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '606'),\n",
       "  ('Min', {}, '20'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '1950.13'),\n",
       "  ('Median', {}, '1621'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '408'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '16239'),\n",
       "  ('Mean', {}, '1524.63'),\n",
       "  ('Median', {}, '1071'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1817'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '12739'),\n",
       "  ('Mean', {}, '266.216'),\n",
       "  ('Median', {}, '138'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '476'),\n",
       "  ('Min', {}, '23'),\n",
       "  ('Max', {}, '7568'),\n",
       "  ('Mean', {}, '347.704'),\n",
       "  ('Median', {}, '154'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '283'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '152236'),\n",
       "  ('Mean', {}, '9397.71'),\n",
       "  ('Median', {}, '5464'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1624'),\n",
       "  ('Min', {}, '68'),\n",
       "  ('Max', {}, '35669'),\n",
       "  ('Mean', {}, '1151.79'),\n",
       "  ('Median', {}, '382'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '394'),\n",
       "  ('Min', {}, '45'),\n",
       "  ('Max', {}, '110042'),\n",
       "  ('Mean', {}, '3451.84'),\n",
       "  ('Median', {}, '791'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '20'),\n",
       "  ('Min', {}, '770'),\n",
       "  ('Max', {}, '2624'),\n",
       "  ('Mean', {}, '1561.95'),\n",
       "  ('Median', {}, '1332'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '963'),\n",
       "  ('Max', {}, '2517'),\n",
       "  ('Mean', {}, '1525.79'),\n",
       "  ('Median', {}, '1260'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '770'),\n",
       "  ('Max', {}, '2624'),\n",
       "  ('Mean', {}, '1646.33'),\n",
       "  ('Median', {}, '1889'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '118'),\n",
       "  ('Min', {}, '149'),\n",
       "  ('Max', {}, '8477'),\n",
       "  ('Mean', {}, '1466.86'),\n",
       "  ('Median', {}, '940'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '408'),\n",
       "  ('Min', {}, '402'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2322.43'),\n",
       "  ('Median', {}, '2028'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '20'),\n",
       "  ('Max', {}, '23'),\n",
       "  ('Mean', {}, '21.6818'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '30'),\n",
       "  ('Min', {}, '505'),\n",
       "  ('Max', {}, '6314'),\n",
       "  ('Mean', {}, '1951.17'),\n",
       "  ('Median', {}, '1603'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '110'),\n",
       "  ('Mean', {}, '81.6667'),\n",
       "  ('Median', {}, '86'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '63'),\n",
       "  ('Max', {}, '132'),\n",
       "  ('Mean', {}, '81.6667'),\n",
       "  ('Median', {}, '77'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '83'),\n",
       "  ('Mean', {}, '74.3182'),\n",
       "  ('Median', {}, '73')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000001375.2', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '13'),\n",
       "  ('Mean', {}, '2.24715'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '65'),\n",
       "  ('Mean', {}, '8.01861'),\n",
       "  ('Median', {}, '6'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '2142'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '12739'),\n",
       "  ('Mean', {}, '286.005'),\n",
       "  ('Median', {}, '145'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '1890'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '110010'),\n",
       "  ('Mean', {}, '1795.23'),\n",
       "  ('Median', {}, '425'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '626'),\n",
       "  ('Min', {}, '19'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '1974.92'),\n",
       "  ('Median', {}, '1678'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '420'),\n",
       "  ('Min', {}, '180'),\n",
       "  ('Max', {}, '16239'),\n",
       "  ('Mean', {}, '1455.1'),\n",
       "  ('Median', {}, '1038'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1852'),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '12739'),\n",
       "  ('Mean', {}, '257.789'),\n",
       "  ('Median', {}, '138'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '564'),\n",
       "  ('Min', {}, '13'),\n",
       "  ('Max', {}, '7568'),\n",
       "  ('Mean', {}, '328.339'),\n",
       "  ('Median', {}, '154'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '290'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '152236'),\n",
       "  ('Mean', {}, '10236'),\n",
       "  ('Median', {}, '5628'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1668'),\n",
       "  ('Min', {}, '68'),\n",
       "  ('Max', {}, '52905'),\n",
       "  ('Mean', {}, '1189.78'),\n",
       "  ('Median', {}, '365'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '466'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '110010'),\n",
       "  ('Mean', {}, '3478.97'),\n",
       "  ('Median', {}, '568'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '23'),\n",
       "  ('Min', {}, '590'),\n",
       "  ('Max', {}, '4525'),\n",
       "  ('Mean', {}, '1549.43'),\n",
       "  ('Median', {}, '1180'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '590'),\n",
       "  ('Max', {}, '4525'),\n",
       "  ('Mean', {}, '1515.24'),\n",
       "  ('Median', {}, '1106'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '770'),\n",
       "  ('Max', {}, '2624'),\n",
       "  ('Mean', {}, '1646.33'),\n",
       "  ('Median', {}, '1889'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '121'),\n",
       "  ('Min', {}, '99'),\n",
       "  ('Max', {}, '8477'),\n",
       "  ('Mean', {}, '1400.26'),\n",
       "  ('Median', {}, '940'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '420'),\n",
       "  ('Min', {}, '402'),\n",
       "  ('Max', {}, '16756'),\n",
       "  ('Mean', {}, '2337.2'),\n",
       "  ('Median', {}, '2042'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '19'),\n",
       "  ('Min', {}, '19'),\n",
       "  ('Max', {}, '23'),\n",
       "  ('Mean', {}, '21.4737'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '36'),\n",
       "  ('Min', {}, '505'),\n",
       "  ('Max', {}, '6314'),\n",
       "  ('Mean', {}, '2293.11'),\n",
       "  ('Median', {}, '1960'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '61'),\n",
       "  ('Max', {}, '107'),\n",
       "  ('Mean', {}, '82.8182'),\n",
       "  ('Median', {}, '86'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '76'),\n",
       "  ('Max', {}, '132'),\n",
       "  ('Mean', {}, '95.3333'),\n",
       "  ('Median', {}, '78'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '27'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '83'),\n",
       "  ('Mean', {}, '74.037'),\n",
       "  ('Median', {}, '73')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510225.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '2.89744'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '18'),\n",
       "  ('Mean', {}, '7.90265'),\n",
       "  ('Median', {}, '7'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '358'),\n",
       "  ('Min', {}, '12'),\n",
       "  ('Max', {}, '4033'),\n",
       "  ('Mean', {}, '248.601'),\n",
       "  ('Median', {}, '154'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '324'),\n",
       "  ('Min', {}, '69'),\n",
       "  ('Max', {}, '36335'),\n",
       "  ('Mean', {}, '1632.85'),\n",
       "  ('Median', {}, '673'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '113'),\n",
       "  ('Min', {}, '21'),\n",
       "  ('Max', {}, '5116'),\n",
       "  ('Mean', {}, '2015.02'),\n",
       "  ('Median', {}, '1870'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '92'),\n",
       "  ('Min', {}, '240'),\n",
       "  ('Max', {}, '3189'),\n",
       "  ('Mean', {}, '1241.15'),\n",
       "  ('Median', {}, '933'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '325'),\n",
       "  ('Min', {}, '12'),\n",
       "  ('Max', {}, '4033'),\n",
       "  ('Mean', {}, '241.431'),\n",
       "  ('Median', {}, '148'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '93'),\n",
       "  ('Min', {}, '28'),\n",
       "  ('Max', {}, '4033'),\n",
       "  ('Mean', {}, '275.516'),\n",
       "  ('Median', {}, '162'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '39'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '36881'),\n",
       "  ('Mean', {}, '11086.5'),\n",
       "  ('Median', {}, '7776'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '299'),\n",
       "  ('Min', {}, '69'),\n",
       "  ('Max', {}, '21374'),\n",
       "  ('Mean', {}, '1558.13'),\n",
       "  ('Median', {}, '646'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '77'),\n",
       "  ('Min', {}, '71'),\n",
       "  ('Max', {}, '36335'),\n",
       "  ('Mean', {}, '1841.68'),\n",
       "  ('Median', {}, '539'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '419'),\n",
       "  ('Max', {}, '2020'),\n",
       "  ('Mean', {}, '970.889'),\n",
       "  ('Median', {}, '1044'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '92'),\n",
       "  ('Min', {}, '526'),\n",
       "  ('Max', {}, '4875'),\n",
       "  ('Mean', {}, '2096.65'),\n",
       "  ('Median', {}, '1956'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '21'),\n",
       "  ('Max', {}, '22'),\n",
       "  ('Mean', {}, '21.5'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '885'),\n",
       "  ('Max', {}, '5116'),\n",
       "  ('Mean', {}, '2602.4'),\n",
       "  ('Median', {}, '2364'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '75'),\n",
       "  ('Mean', {}, '73.5'),\n",
       "  ('Median', {}, '75')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510235.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '2.92593'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '18'),\n",
       "  ('Mean', {}, '8.02532'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '503'),\n",
       "  ('Min', {}, '12'),\n",
       "  ('Max', {}, '4033'),\n",
       "  ('Mean', {}, '248.541'),\n",
       "  ('Median', {}, '155'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '460'),\n",
       "  ('Min', {}, '69'),\n",
       "  ('Max', {}, '36335'),\n",
       "  ('Mean', {}, '1682.65'),\n",
       "  ('Median', {}, '652'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '158'),\n",
       "  ('Min', {}, '21'),\n",
       "  ('Max', {}, '5116'),\n",
       "  ('Mean', {}, '2097.47'),\n",
       "  ('Median', {}, '2056'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '126'),\n",
       "  ('Min', {}, '240'),\n",
       "  ('Max', {}, '3189'),\n",
       "  ('Mean', {}, '1266.02'),\n",
       "  ('Median', {}, '1035'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '451'),\n",
       "  ('Min', {}, '12'),\n",
       "  ('Max', {}, '4033'),\n",
       "  ('Mean', {}, '243.96'),\n",
       "  ('Median', {}, '142'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '142'),\n",
       "  ('Min', {}, '28'),\n",
       "  ('Max', {}, '4033'),\n",
       "  ('Mean', {}, '264.437'),\n",
       "  ('Median', {}, '171'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '54'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '36881'),\n",
       "  ('Mean', {}, '11802.4'),\n",
       "  ('Median', {}, '9771'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '419'),\n",
       "  ('Min', {}, '69'),\n",
       "  ('Max', {}, '21374'),\n",
       "  ('Mean', {}, '1586.19'),\n",
       "  ('Median', {}, '619'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '118'),\n",
       "  ('Min', {}, '71'),\n",
       "  ('Max', {}, '36335'),\n",
       "  ('Mean', {}, '1686.7'),\n",
       "  ('Median', {}, '448'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '2050'),\n",
       "  ('Max', {}, '2050'),\n",
       "  ('Mean', {}, '2050'),\n",
       "  ('Median', {}, '2050'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '2050'),\n",
       "  ('Max', {}, '2050'),\n",
       "  ('Mean', {}, '2050'),\n",
       "  ('Median', {}, '2050'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '16'),\n",
       "  ('Min', {}, '419'),\n",
       "  ('Max', {}, '2184'),\n",
       "  ('Mean', {}, '1075.81'),\n",
       "  ('Median', {}, '1009'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '126'),\n",
       "  ('Min', {}, '526'),\n",
       "  ('Max', {}, '5097'),\n",
       "  ('Mean', {}, '2208.09'),\n",
       "  ('Median', {}, '2149'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '21'),\n",
       "  ('Max', {}, '22'),\n",
       "  ('Mean', {}, '21.5'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '885'),\n",
       "  ('Max', {}, '5116'),\n",
       "  ('Mean', {}, '2566.14'),\n",
       "  ('Median', {}, '2364'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '72'),\n",
       "  ('Max', {}, '75'),\n",
       "  ('Mean', {}, '73.5'),\n",
       "  ('Median', {}, '75')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509915.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.75'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.33333'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '95'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '197.316'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '90'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '1889.01'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '21'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1436.57'),\n",
       "  ('Median', {}, '1522'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '19'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '991.263'),\n",
       "  ('Median', {}, '1029'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '91'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '195.242'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16787'),\n",
       "  ('Mean', {}, '12227.3'),\n",
       "  ('Median', {}, '14312'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '87'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '1934.48'),\n",
       "  ('Median', {}, '1304'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '19'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1509.32'),\n",
       "  ('Median', {}, '1573')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509925.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.5'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.16667'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.242'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '57'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9892'),\n",
       "  ('Mean', {}, '1948.54'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1523.5'),\n",
       "  ('Median', {}, '1592'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '669'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1056.6'),\n",
       "  ('Median', {}, '1047'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '204.672'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16787'),\n",
       "  ('Mean', {}, '12586'),\n",
       "  ('Median', {}, '14521'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '54'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9892'),\n",
       "  ('Mean', {}, '2025.11'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1679.1'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509935.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '2'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.71429'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '59'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '688'),\n",
       "  ('Mean', {}, '210.644'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9892'),\n",
       "  ('Mean', {}, '2216.9'),\n",
       "  ('Median', {}, '1514'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1573.36'),\n",
       "  ('Median', {}, '1582'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '970.286'),\n",
       "  ('Median', {}, '1029'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '59'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '688'),\n",
       "  ('Mean', {}, '210.644'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '7'),\n",
       "  ('Min', {}, '10911'),\n",
       "  ('Max', {}, '16733'),\n",
       "  ('Mean', {}, '14087.1'),\n",
       "  ('Median', {}, '14345'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9892'),\n",
       "  ('Mean', {}, '2216.9'),\n",
       "  ('Median', {}, '1514'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1573.36'),\n",
       "  ('Median', {}, '1582')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509945.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.375'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.18182'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.613'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '56'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '1850.05'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1524.82'),\n",
       "  ('Median', {}, '1605'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '915'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1156'),\n",
       "  ('Median', {}, '1134'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '205.069'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16751'),\n",
       "  ('Mean', {}, '12591.4'),\n",
       "  ('Median', {}, '14530'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '53'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '1922.49'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1698'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509955.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.72727'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '6.89474'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '84'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '204.393'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '77'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '7715'),\n",
       "  ('Mean', {}, '1864.09'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '19'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1467.21'),\n",
       "  ('Median', {}, '1573'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '980.824'),\n",
       "  ('Median', {}, '1029'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '80'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '202.362'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '245'),\n",
       "  ('Median', {}, '269'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '1644'),\n",
       "  ('Max', {}, '15237'),\n",
       "  ('Mean', {}, '11532.4'),\n",
       "  ('Median', {}, '14312'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '74'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '7715'),\n",
       "  ('Mean', {}, '1916.54'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1552.12'),\n",
       "  ('Median', {}, '1582')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509965.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '3'),\n",
       "  ('Mean', {}, '1.45455'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.0625'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '81'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '191.099'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '74'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '1841.31'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '16'),\n",
       "  ('Min', {}, '536'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1388.94'),\n",
       "  ('Median', {}, '1582'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '13'),\n",
       "  ('Min', {}, '828'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1073.31'),\n",
       "  ('Median', {}, '1047'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '74'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '188.716'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '7'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '216.286'),\n",
       "  ('Median', {}, '197'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16787'),\n",
       "  ('Mean', {}, '11811.2'),\n",
       "  ('Median', {}, '14015'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '69'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '1856.09'),\n",
       "  ('Median', {}, '1304'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '5'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '5470'),\n",
       "  ('Mean', {}, '1637.4'),\n",
       "  ('Median', {}, '849'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '13'),\n",
       "  ('Min', {}, '1101'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1553.54'),\n",
       "  ('Median', {}, '1592'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '1'),\n",
       "  ('Min', {}, '536'),\n",
       "  ('Max', {}, '536'),\n",
       "  ('Mean', {}, '536'),\n",
       "  ('Median', {}, '536')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509975.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.85714'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.76923'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '59'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '688'),\n",
       "  ('Mean', {}, '211.017'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '57'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9891'),\n",
       "  ('Mean', {}, '2103.23'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '13'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1578.23'),\n",
       "  ('Median', {}, '1582'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '13'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1008.46'),\n",
       "  ('Median', {}, '1029'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '59'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '688'),\n",
       "  ('Mean', {}, '211.017'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '7'),\n",
       "  ('Min', {}, '10951'),\n",
       "  ('Max', {}, '16752'),\n",
       "  ('Mean', {}, '13916.7'),\n",
       "  ('Median', {}, '14344'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '57'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9891'),\n",
       "  ('Mean', {}, '2103.23'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '13'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1578.23'),\n",
       "  ('Median', {}, '1582')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509985.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.375'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.18182'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.613'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '56'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '1849.8'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1524.82'),\n",
       "  ('Median', {}, '1605'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '915'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1156'),\n",
       "  ('Median', {}, '1134'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '205.069'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16751'),\n",
       "  ('Mean', {}, '12589.6'),\n",
       "  ('Median', {}, '14519'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '53'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '1922.23'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1698'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000509995.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.88889'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.76471'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '75'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '694'),\n",
       "  ('Mean', {}, '200.12'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '2023.41'),\n",
       "  ('Median', {}, '1476'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1524.71'),\n",
       "  ('Median', {}, '1573'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '979.941'),\n",
       "  ('Median', {}, '1017'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '75'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '694'),\n",
       "  ('Mean', {}, '200.12'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '9465'),\n",
       "  ('Max', {}, '16787'),\n",
       "  ('Mean', {}, '13526'),\n",
       "  ('Median', {}, '14313'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '2023.41'),\n",
       "  ('Median', {}, '1476'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1524.71'),\n",
       "  ('Median', {}, '1573')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510005.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.375'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.18182'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.613'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '56'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9889'),\n",
       "  ('Mean', {}, '1849.36'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1524.82'),\n",
       "  ('Median', {}, '1605'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '915'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1156'),\n",
       "  ('Median', {}, '1134'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '205.069'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16765'),\n",
       "  ('Mean', {}, '12586.6'),\n",
       "  ('Median', {}, '14520'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '53'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9889'),\n",
       "  ('Mean', {}, '1921.75'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1698'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510015.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.72727'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '6.89474'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '84'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '210.167'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '77'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '7715'),\n",
       "  ('Mean', {}, '1875.66'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '19'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1492.89'),\n",
       "  ('Median', {}, '1581'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '962.471'),\n",
       "  ('Median', {}, '1017'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '80'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '208.45'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '15868'),\n",
       "  ('Mean', {}, '11657.5'),\n",
       "  ('Median', {}, '14312'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '74'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '7715'),\n",
       "  ('Mean', {}, '1928.58'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1580.82'),\n",
       "  ('Median', {}, '1583')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510025.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.375'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.18182'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.613'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '56'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '1849.16'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1524.82'),\n",
       "  ('Median', {}, '1605'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '915'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1156'),\n",
       "  ('Median', {}, '1134'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '205.069'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16751'),\n",
       "  ('Mean', {}, '12585.1'),\n",
       "  ('Median', {}, '14514'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '53'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '1921.55'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1698'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510035.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.88889'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '6'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.76471'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '75'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '688'),\n",
       "  ('Mean', {}, '193.573'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '2020.36'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1495.94'),\n",
       "  ('Median', {}, '1522'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '979.941'),\n",
       "  ('Median', {}, '1017'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '75'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '688'),\n",
       "  ('Mean', {}, '193.573'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '9461'),\n",
       "  ('Max', {}, '16787'),\n",
       "  ('Mean', {}, '13446.7'),\n",
       "  ('Median', {}, '14313'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '2020.36'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1495.94'),\n",
       "  ('Median', {}, '1522')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510045.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '2.4'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '5.54167'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '76'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '221.921'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '70'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2227.69'),\n",
       "  ('Median', {}, '1420'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '24'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1684.92'),\n",
       "  ('Median', {}, '1723'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '537'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '901.773'),\n",
       "  ('Median', {}, '912'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '72'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '220.667'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '17186'),\n",
       "  ('Mean', {}, '12181.9'),\n",
       "  ('Median', {}, '14529'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '67'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2301.9'),\n",
       "  ('Median', {}, '1428'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '821'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1770.32'),\n",
       "  ('Median', {}, '1826')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510055.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.33333'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '6.125'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '40'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '205.225'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '35'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '5467'),\n",
       "  ('Mean', {}, '1581'),\n",
       "  ('Median', {}, '934'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1433.12'),\n",
       "  ('Median', {}, '1592'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '1026'),\n",
       "  ('Max', {}, '1335'),\n",
       "  ('Mean', {}, '1134'),\n",
       "  ('Median', {}, '1134'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '36'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '200.861'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '14531'),\n",
       "  ('Mean', {}, '9614.83'),\n",
       "  ('Median', {}, '12048'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '32'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '5467'),\n",
       "  ('Mean', {}, '1675.75'),\n",
       "  ('Median', {}, '999'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '6'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1662.33'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510065.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.5'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.16667'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.242'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '57'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '1948.89'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1523.5'),\n",
       "  ('Median', {}, '1592'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '669'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1056.6'),\n",
       "  ('Median', {}, '1047'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '204.672'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16789'),\n",
       "  ('Mean', {}, '12588.8'),\n",
       "  ('Median', {}, '14530'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '54'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9890'),\n",
       "  ('Mean', {}, '2025.48'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1679.1'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510075.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '2.4'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '5.54167'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '76'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '221.921'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '70'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2216.06'),\n",
       "  ('Median', {}, '1401'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '24'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1684.92'),\n",
       "  ('Median', {}, '1723'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '537'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '901.773'),\n",
       "  ('Median', {}, '912'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '72'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '220.667'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '17186'),\n",
       "  ('Mean', {}, '12100.6'),\n",
       "  ('Median', {}, '14528'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '67'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2289.75'),\n",
       "  ('Median', {}, '1420'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '821'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1770.32'),\n",
       "  ('Median', {}, '1826')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510085.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '2.33333'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '6.17143'),\n",
       "  ('Median', {}, '7'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '117'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '202.171'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '111'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2122.87'),\n",
       "  ('Median', {}, '1420'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '35'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1564.23'),\n",
       "  ('Median', {}, '1573'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '33'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '884.364'),\n",
       "  ('Median', {}, '912'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '113'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '200.673'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '15'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '17186'),\n",
       "  ('Mean', {}, '12135.1'),\n",
       "  ('Median', {}, '14159'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '108'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2166'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '33'),\n",
       "  ('Min', {}, '821'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1613.85'),\n",
       "  ('Median', {}, '1582')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510095.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '2.5'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '5.6'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '76'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '221.632'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '71'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2301.82'),\n",
       "  ('Median', {}, '1420'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '25'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1677.92'),\n",
       "  ('Median', {}, '1614'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '23'),\n",
       "  ('Min', {}, '537'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '883.174'),\n",
       "  ('Median', {}, '864'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '72'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '220.361'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '17186'),\n",
       "  ('Mean', {}, '12180.8'),\n",
       "  ('Median', {}, '14519'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '68'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2378.21'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '23'),\n",
       "  ('Min', {}, '821'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1759'),\n",
       "  ('Median', {}, '1723')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510105.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '3'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '6'),\n",
       "  ('Median', {}, '6'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '224.945'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '72'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2515.99'),\n",
       "  ('Median', {}, '1514'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '27'),\n",
       "  ('Min', {}, '821'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1692.26'),\n",
       "  ('Median', {}, '1583'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '27'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '841'),\n",
       "  ('Median', {}, '828'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '224.945'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '3950'),\n",
       "  ('Max', {}, '17186'),\n",
       "  ('Mean', {}, '13325.7'),\n",
       "  ('Median', {}, '14341'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '72'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2515.99'),\n",
       "  ('Median', {}, '1514'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '27'),\n",
       "  ('Min', {}, '821'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1692.26'),\n",
       "  ('Median', {}, '1583')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510125.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '8'),\n",
       "  ('Mean', {}, '2.4'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '5.45833'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '74'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '226.311'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '68'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2272.18'),\n",
       "  ('Median', {}, '1428'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '24'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1684.92'),\n",
       "  ('Median', {}, '1723'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '537'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '901.773'),\n",
       "  ('Median', {}, '912'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '70'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '1772'),\n",
       "  ('Mean', {}, '225.271'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '17186'),\n",
       "  ('Mean', {}, '12027.2'),\n",
       "  ('Median', {}, '14529'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '65'),\n",
       "  ('Min', {}, '77'),\n",
       "  ('Max', {}, '13683'),\n",
       "  ('Mean', {}, '2350.72'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '22'),\n",
       "  ('Min', {}, '821'),\n",
       "  ('Max', {}, '2476'),\n",
       "  ('Mean', {}, '1770.32'),\n",
       "  ('Median', {}, '1826')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510135.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.4'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.28571'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '78'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '206.141'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '71'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9889'),\n",
       "  ('Mean', {}, '1863.48'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1536.86'),\n",
       "  ('Median', {}, '1602'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '669'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1050.75'),\n",
       "  ('Median', {}, '1047'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '74'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '204.068'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16769'),\n",
       "  ('Mean', {}, '12516.1'),\n",
       "  ('Median', {}, '14535'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '68'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9889'),\n",
       "  ('Mean', {}, '1920.53'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1668.75'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510145.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.375'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.18182'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.613'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '56'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9892'),\n",
       "  ('Mean', {}, '1833.68'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1524.82'),\n",
       "  ('Median', {}, '1605'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '915'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1156'),\n",
       "  ('Median', {}, '1134'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '205.069'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16768'),\n",
       "  ('Mean', {}, '12476.5'),\n",
       "  ('Median', {}, '14514'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '53'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9892'),\n",
       "  ('Mean', {}, '1905.19'),\n",
       "  ('Median', {}, '1273'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1698'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510155.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.72727'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.26316'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '87'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '209.793'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '82'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '2009.87'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '19'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1492.89'),\n",
       "  ('Median', {}, '1581'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '962.471'),\n",
       "  ('Median', {}, '1017'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '83'),\n",
       "  ('Min', {}, '34'),\n",
       "  ('Max', {}, '774'),\n",
       "  ('Mean', {}, '208.12'),\n",
       "  ('Median', {}, '104'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '11'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16787'),\n",
       "  ('Mean', {}, '12692.8'),\n",
       "  ('Median', {}, '14342'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '79'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9888'),\n",
       "  ('Mean', {}, '2064.53'),\n",
       "  ('Median', {}, '1476'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1580.82'),\n",
       "  ('Median', {}, '1583')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510165.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '2'),\n",
       "  ('Mean', {}, '1.5'),\n",
       "  ('Median', {}, '2'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '3'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.16667'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '62'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '207.242'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '57'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '9889'),\n",
       "  ('Mean', {}, '1945.98'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '12'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1523.5'),\n",
       "  ('Median', {}, '1592'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '669'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '1056.6'),\n",
       "  ('Median', {}, '1047'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '58'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '686'),\n",
       "  ('Mean', {}, '204.672'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '4'),\n",
       "  ('Min', {}, '43'),\n",
       "  ('Max', {}, '470'),\n",
       "  ('Mean', {}, '244.5'),\n",
       "  ('Median', {}, '268'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '8'),\n",
       "  ('Min', {}, '1643'),\n",
       "  ('Max', {}, '16765'),\n",
       "  ('Mean', {}, '12568'),\n",
       "  ('Median', {}, '14527'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '54'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '9889'),\n",
       "  ('Mean', {}, '2022.41'),\n",
       "  ('Median', {}, '1464'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '3'),\n",
       "  ('Min', {}, '84'),\n",
       "  ('Max', {}, '849'),\n",
       "  ('Mean', {}, '570.333'),\n",
       "  ('Median', {}, '778'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '710'),\n",
       "  ('Max', {}, '781'),\n",
       "  ('Mean', {}, '745.5'),\n",
       "  ('Median', {}, '781'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '1477'),\n",
       "  ('Max', {}, '1986'),\n",
       "  ('Mean', {}, '1679.1'),\n",
       "  ('Median', {}, '1614')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000510175.1', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '5'),\n",
       "  ('Mean', {}, '1.88889'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '5'),\n",
       "  ('Max', {}, '9'),\n",
       "  ('Mean', {}, '7.41176'),\n",
       "  ('Median', {}, '8'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '694'),\n",
       "  ('Mean', {}, '198.123'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '69'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '7716'),\n",
       "  ('Mean', {}, '1891.99'),\n",
       "  ('Median', {}, '1476'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1524.71'),\n",
       "  ('Median', {}, '1573'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '321'),\n",
       "  ('Max', {}, '1368'),\n",
       "  ('Mean', {}, '979.941'),\n",
       "  ('Median', {}, '1017'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '73'),\n",
       "  ('Min', {}, '36'),\n",
       "  ('Max', {}, '694'),\n",
       "  ('Mean', {}, '198.123'),\n",
       "  ('Median', {}, '105'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '9'),\n",
       "  ('Min', {}, '8713'),\n",
       "  ('Max', {}, '15021'),\n",
       "  ('Mean', {}, '12624.4'),\n",
       "  ('Median', {}, '14015'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '69'),\n",
       "  ('Min', {}, '98'),\n",
       "  ('Max', {}, '7716'),\n",
       "  ('Mean', {}, '1891.99'),\n",
       "  ('Median', {}, '1476'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '17'),\n",
       "  ('Min', {}, '979'),\n",
       "  ('Max', {}, '1877'),\n",
       "  ('Mean', {}, '1524.71'),\n",
       "  ('Median', {}, '1573')],\n",
       " [('AssemblyStats',\n",
       "   {'Accession': 'GCF_000005045.25', 'Type': 'assembly unit'},\n",
       "   '\\n    '),\n",
       "  ('TranscriptsPerGene', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '33'),\n",
       "  ('Mean', {}, '2.14199'),\n",
       "  ('Median', {}, '1'),\n",
       "  ('ExonsPerTranscript', {}, '\\n      '),\n",
       "  ('Min', {}, '1'),\n",
       "  ('Max', {}, '93'),\n",
       "  ('Mean', {}, '7.87595'),\n",
       "  ('Median', {}, '5'),\n",
       "  ('LengthStats', {'category': 'All exons'}, '\\n      '),\n",
       "  ('Count', {}, '8186'),\n",
       "  ('Min', {}, '7'),\n",
       "  ('Max', {}, '14433'),\n",
       "  ('Mean', {}, '366.997'),\n",
       "  ('Median', {}, '138'),\n",
       "  ('LengthStats', {'category': 'All introns'}, '\\n      '),\n",
       "  ('Count', {}, '7161'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '334517'),\n",
       "  ('Mean', {}, '4315.23'),\n",
       "  ('Median', {}, '1113'),\n",
       "  ('LengthStats', {'category': 'All transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '2140'),\n",
       "  ('Min', {}, '17'),\n",
       "  ('Max', {}, '21046'),\n",
       "  ('Mean', {}, '2885.79'),\n",
       "  ('Median', {}, '2234'),\n",
       "  ('LengthStats', {'category': 'CDSs'}, '\\n      '),\n",
       "  ('Count', {}, '1382'),\n",
       "  ('Min', {}, '132'),\n",
       "  ('Max', {}, '16965'),\n",
       "  ('Mean', {}, '1728.69'),\n",
       "  ('Median', {}, '1284'),\n",
       "  ('LengthStats', {'category': 'Exons in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '6730'),\n",
       "  ('Min', {}, '7'),\n",
       "  ('Max', {}, '14433'),\n",
       "  ('Mean', {}, '339.759'),\n",
       "  ('Median', {}, '133'),\n",
       "  ('LengthStats', {'category': 'Exons in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '2042'),\n",
       "  ('Min', {}, '9'),\n",
       "  ('Max', {}, '10558'),\n",
       "  ('Mean', {}, '427.424'),\n",
       "  ('Median', {}, '153'),\n",
       "  ('LengthStats', {'category': 'Genes'}, '\\n      '),\n",
       "  ('Count', {}, '996'),\n",
       "  ('Min', {}, '51'),\n",
       "  ('Max', {}, '650936'),\n",
       "  ('Mean', {}, '25456.7'),\n",
       "  ('Median', {}, '8349'),\n",
       "  ('LengthStats', {'category': 'Introns in coding models'}, '\\n      '),\n",
       "  ('Count', {}, '6034'),\n",
       "  ('Min', {}, '30'),\n",
       "  ('Max', {}, '334517'),\n",
       "  ('Mean', {}, '3872.72'),\n",
       "  ('Median', {}, '1015'),\n",
       "  ('LengthStats', {'category': 'Introns in non-coding models'}, '\\n      '),\n",
       "  ('Count', {}, '1642'),\n",
       "  ('Min', {}, '38'),\n",
       "  ('Max', {}, '112483'),\n",
       "  ('Mean', {}, '5694.09'),\n",
       "  ('Median', {}, '1571'),\n",
       "  ('LengthStats', {'category': 'Single-exon transcripts'}, '\\n      '),\n",
       "  ('Count', {}, '118'),\n",
       "  ('Min', {}, '452'),\n",
       "  ('Max', {}, '18485'),\n",
       "  ('Mean', {}, '3601.76'),\n",
       "  ('Median', {}, '2485'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '104'),\n",
       "  ('Min', {}, '452'),\n",
       "  ('Max', {}, '18485'),\n",
       "  ('Mean', {}, '3517.5'),\n",
       "  ('Median', {}, '2462'),\n",
       "  ('LengthStats',\n",
       "   {'category': 'Single-exon transcripts in non-coding models'},\n",
       "   '\\n      '),\n",
       "  ('Count', {}, '14'),\n",
       "  ('Min', {}, '829'),\n",
       "  ('Max', {}, '12682'),\n",
       "  ('Mean', {}, '4227.71'),\n",
       "  ('Median', {}, '2879'),\n",
       "  ('LengthStats', {'category': 'guide_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '266'),\n",
       "  ('Max', {}, '278'),\n",
       "  ('Mean', {}, '272'),\n",
       "  ('Median', {}, '278'),\n",
       "  ('LengthStats', {'category': 'lncRNA'}, '\\n      '),\n",
       "  ('Count', {}, '475'),\n",
       "  ('Min', {}, '137'),\n",
       "  ('Max', {}, '13054'),\n",
       "  ('Mean', {}, '1834.45'),\n",
       "  ('Median', {}, '1308'),\n",
       "  ('LengthStats', {'category': 'mRNA'}, '\\n      '),\n",
       "  ('Count', {}, '1382'),\n",
       "  ('Min', {}, '234'),\n",
       "  ('Max', {}, '21046'),\n",
       "  ('Mean', {}, '3470.96'),\n",
       "  ('Median', {}, '2585'),\n",
       "  ('LengthStats', {'category': 'miRNA'}, '\\n      '),\n",
       "  ('Count', {}, '83'),\n",
       "  ('Min', {}, '17'),\n",
       "  ('Max', {}, '26'),\n",
       "  ('Mean', {}, '21.6386'),\n",
       "  ('Median', {}, '22'),\n",
       "  ('LengthStats', {'category': 'misc_RNA'}, '\\n      '),\n",
       "  ('Count', {}, '161'),\n",
       "  ('Min', {}, '173'),\n",
       "  ('Max', {}, '12682'),\n",
       "  ('Mean', {}, '2863.29'),\n",
       "  ('Median', {}, '2760'),\n",
       "  ('LengthStats', {'category': 'premsg'}, '\\n      '),\n",
       "  ('Count', {}, '57'),\n",
       "  ('Min', {}, '51'),\n",
       "  ('Max', {}, '138'),\n",
       "  ('Mean', {}, '83.7368'),\n",
       "  ('Median', {}, '80'),\n",
       "  ('LengthStats', {'category': 'rRNA'}, '\\n      '),\n",
       "  ('Count', {}, '25'),\n",
       "  ('Min', {}, '121'),\n",
       "  ('Max', {}, '13351'),\n",
       "  ('Mean', {}, '1715.6'),\n",
       "  ('Median', {}, '121'),\n",
       "  ('LengthStats', {'category': 'snoRNA'}, '\\n      '),\n",
       "  ('Count', {}, '10'),\n",
       "  ('Min', {}, '59'),\n",
       "  ('Max', {}, '197'),\n",
       "  ('Mean', {}, '98.2'),\n",
       "  ('Median', {}, '75'),\n",
       "  ('LengthStats', {'category': 'tRNA'}, '\\n      '),\n",
       "  ('Count', {}, '2'),\n",
       "  ('Min', {}, '82'),\n",
       "  ('Max', {}, '82'),\n",
       "  ('Mean', {}, '82'),\n",
       "  ('Median', {}, '82')]]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "feature_stats_collection = []\n",
    "for i in root[3]:\n",
    "    tags = []\n",
    "    text = []\n",
    "    attributes = []\n",
    "    for element in i.iter():\n",
    "        tags.append(element.tag)\n",
    "        text.append(element.text)\n",
    "        attributes.append(element.attrib)\n",
    "    thing = list(zip(tags,attributes,text))\n",
    "    feature_stats_collection.append(thing)\n",
    "print(len(feature_stats_collection))\n",
    "feature_stats_collection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'GCF_000001405.39'"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "human_accession = full_assembly_collection[1][0][2]\n",
    "human_accession"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "From this 'human_accession', at the ncbi online database located at https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.39/, the full sequence report can be pulled down into a text file seen below. This information contains information about genetic information located on chromosomes, in other words, it's a type of dna data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "import csv\n",
    "txt_file_dna = r\"GCF_000001405.39_GRCh38.p13_assembly_report.txt\"\n",
    "csv_file_dna = r\"GCF_000001405.39_GRCh38.p13_assembly_report.csv\"\n",
    "in_txt_dna = csv.reader(open(txt_file_dna, \"r\", encoding='utf-8'), delimiter = '\\t')\n",
    "out_csv_dna = csv.writer(open(csv_file_dna, 'w'))\n",
    "out_csv_dna.writerows(in_txt_dna)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Assembly name:  GRCh38.p13</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>Description:    Genome Reference Consortium Hu...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>Organism name:  Homo sapiens (human)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      <td>Taxid:          9606</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      <td>BioProject:     PRJNA31257</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      <td>Submitter:      Genome Reference Consortium</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      <td>Date:           2019-02-28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      <td>Assembly type:  haploid-with-alt-loci</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      <td>Release type:   patch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      <td>Assembly level: Chromosome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      <td>Genome representation: full</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      <td>RefSeq category: Reference Genome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      <td>GenBank assembly accession: GCA_000001405.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      <td>RefSeq assembly accession: GCF_000001405.39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      <td>RefSeq assembly and GenBank assemblies identic...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>31</td>\n",
       "      <td>Assembly-Units:</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           Assembly name:  GRCh38.p13\n",
       "1   Description:    Genome Reference Consortium Hu...\n",
       "3                Organism name:  Homo sapiens (human)\n",
       "5                                Taxid:          9606\n",
       "7                          BioProject:     PRJNA31257\n",
       "9         Submitter:      Genome Reference Consortium\n",
       "11                         Date:           2019-02-28\n",
       "13              Assembly type:  haploid-with-alt-loci\n",
       "15                              Release type:   patch\n",
       "17                         Assembly level: Chromosome\n",
       "19                        Genome representation: full\n",
       "21                  RefSeq category: Reference Genome\n",
       "23       GenBank assembly accession: GCA_000001405.28\n",
       "25        RefSeq assembly accession: GCF_000001405.39\n",
       "27  RefSeq assembly and GenBank assemblies identic...\n",
       "31                                    Assembly-Units:"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "to_skip_info = []\n",
    "for i in range(34,1406):\n",
    "    to_skip_info.append(i)\n",
    "information_about_dna_data = pd.read_csv('GCF_000001405.39_GRCh38.p13_assembly_report_fixed.csv', skiprows=(to_skip_info), header=0)\n",
    "pd.DataFrame(information_about_dna_data['Assembly name:  GRCh38.p13'].dropna(axis=0))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "dna_data = pd.read_csv('GCF_000001405.39_GRCh38.p13_assembly_report_fixed.csv', header=124)\n",
    "dna_data = dna_data.dropna(axis=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th># Sequence-Name</th>\n",
       "      <th>Sequence-Role</th>\n",
       "      <th>Assigned-Molecule</th>\n",
       "      <th>Assigned-Molecule-Location/Type</th>\n",
       "      <th>GenBank-Accn</th>\n",
       "      <th>Relationship</th>\n",
       "      <th>RefSeq-Accn</th>\n",
       "      <th>Assembly-Unit</th>\n",
       "      <th>Sequence-Length</th>\n",
       "      <th>UCSC-style-name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>395</td>\n",
       "      <td>HG1342_HG2282_PATCH</td>\n",
       "      <td>fix-patch</td>\n",
       "      <td>1</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KQ031383.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NW_012132914.1</td>\n",
       "      <td>PATCHES</td>\n",
       "      <td>467143.0</td>\n",
       "      <td>na</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1259</td>\n",
       "      <td>HSCHR19KIR_FH05_B_HAP_CTG3_1</td>\n",
       "      <td>alt-scaffold</td>\n",
       "      <td>19</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KI270921.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NT_187675.1</td>\n",
       "      <td>ALT_REF_LOCI_27</td>\n",
       "      <td>282224.0</td>\n",
       "      <td>chr19_KI270921v1_alt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>731</td>\n",
       "      <td>HSCHR22_6_CTG1</td>\n",
       "      <td>novel-patch</td>\n",
       "      <td>22</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KQ458387.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NW_014040930.1</td>\n",
       "      <td>PATCHES</td>\n",
       "      <td>155930.0</td>\n",
       "      <td>na</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>973</td>\n",
       "      <td>HSCHR12_5_CTG2_1</td>\n",
       "      <td>alt-scaffold</td>\n",
       "      <td>12</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KI270833.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NT_187589.1</td>\n",
       "      <td>ALT_REF_LOCI_1</td>\n",
       "      <td>76061.0</td>\n",
       "      <td>chr12_KI270833v1_alt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>873</td>\n",
       "      <td>HSCHR6_1_CTG6</td>\n",
       "      <td>alt-scaffold</td>\n",
       "      <td>6</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KI270799.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NT_187554.1</td>\n",
       "      <td>ALT_REF_LOCI_1</td>\n",
       "      <td>152148.0</td>\n",
       "      <td>chr6_KI270799v1_alt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1165</td>\n",
       "      <td>HSCHR17_2_CTG5</td>\n",
       "      <td>alt-scaffold</td>\n",
       "      <td>17</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KI270908.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NT_187663.1</td>\n",
       "      <td>ALT_REF_LOCI_2</td>\n",
       "      <td>1423190.0</td>\n",
       "      <td>chr17_KI270908v1_alt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>643</td>\n",
       "      <td>HG926_PATCH</td>\n",
       "      <td>fix-patch</td>\n",
       "      <td>16</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KV880768.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NW_017852933.1</td>\n",
       "      <td>PATCHES</td>\n",
       "      <td>1927115.0</td>\n",
       "      <td>na</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>499</td>\n",
       "      <td>HG2057_PATCH</td>\n",
       "      <td>fix-patch</td>\n",
       "      <td>6</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KZ208911.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NW_018654713.1</td>\n",
       "      <td>PATCHES</td>\n",
       "      <td>242796.0</td>\n",
       "      <td>na</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1025</td>\n",
       "      <td>HSCHR16_3_CTG1</td>\n",
       "      <td>alt-scaffold</td>\n",
       "      <td>16</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>KI270855.1</td>\n",
       "      <td>=</td>\n",
       "      <td>NT_187608.1</td>\n",
       "      <td>ALT_REF_LOCI_1</td>\n",
       "      <td>232857.0</td>\n",
       "      <td>chr16_KI270855v1_alt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>781</td>\n",
       "      <td>HSCHR2_2_CTG7_2</td>\n",
       "      <td>alt-scaffold</td>\n",
       "      <td>2</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>GL582966.2</td>\n",
       "      <td>=</td>\n",
       "      <td>NW_003571033.2</td>\n",
       "      <td>ALT_REF_LOCI_1</td>\n",
       "      <td>96131.0</td>\n",
       "      <td>chr2_GL582966v2_alt</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                   # Sequence-Name Sequence-Role Assigned-Molecule  \\\n",
       "395            HG1342_HG2282_PATCH     fix-patch                 1   \n",
       "1259  HSCHR19KIR_FH05_B_HAP_CTG3_1  alt-scaffold                19   \n",
       "731                 HSCHR22_6_CTG1   novel-patch                22   \n",
       "973               HSCHR12_5_CTG2_1  alt-scaffold                12   \n",
       "873                  HSCHR6_1_CTG6  alt-scaffold                 6   \n",
       "1165                HSCHR17_2_CTG5  alt-scaffold                17   \n",
       "643                    HG926_PATCH     fix-patch                16   \n",
       "499                   HG2057_PATCH     fix-patch                 6   \n",
       "1025                HSCHR16_3_CTG1  alt-scaffold                16   \n",
       "781                HSCHR2_2_CTG7_2  alt-scaffold                 2   \n",
       "\n",
       "     Assigned-Molecule-Location/Type GenBank-Accn Relationship  \\\n",
       "395                       Chromosome   KQ031383.1            =   \n",
       "1259                      Chromosome   KI270921.1            =   \n",
       "731                       Chromosome   KQ458387.1            =   \n",
       "973                       Chromosome   KI270833.1            =   \n",
       "873                       Chromosome   KI270799.1            =   \n",
       "1165                      Chromosome   KI270908.1            =   \n",
       "643                       Chromosome   KV880768.1            =   \n",
       "499                       Chromosome   KZ208911.1            =   \n",
       "1025                      Chromosome   KI270855.1            =   \n",
       "781                       Chromosome   GL582966.2            =   \n",
       "\n",
       "         RefSeq-Accn    Assembly-Unit  Sequence-Length       UCSC-style-name  \n",
       "395   NW_012132914.1          PATCHES         467143.0                    na  \n",
       "1259     NT_187675.1  ALT_REF_LOCI_27         282224.0  chr19_KI270921v1_alt  \n",
       "731   NW_014040930.1          PATCHES         155930.0                    na  \n",
       "973      NT_187589.1   ALT_REF_LOCI_1          76061.0  chr12_KI270833v1_alt  \n",
       "873      NT_187554.1   ALT_REF_LOCI_1         152148.0   chr6_KI270799v1_alt  \n",
       "1165     NT_187663.1   ALT_REF_LOCI_2        1423190.0  chr17_KI270908v1_alt  \n",
       "643   NW_017852933.1          PATCHES        1927115.0                    na  \n",
       "499   NW_018654713.1          PATCHES         242796.0                    na  \n",
       "1025     NT_187608.1   ALT_REF_LOCI_1         232857.0  chr16_KI270855v1_alt  \n",
       "781   NW_003571033.2   ALT_REF_LOCI_1          96131.0   chr2_GL582966v2_alt  "
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dna_data.sample(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Int64Index: 640 entries, 1 to 1279\n",
      "Data columns (total 10 columns):\n",
      "# Sequence-Name                    640 non-null object\n",
      "Sequence-Role                      640 non-null object\n",
      "Assigned-Molecule                  640 non-null object\n",
      "Assigned-Molecule-Location/Type    640 non-null object\n",
      "GenBank-Accn                       640 non-null object\n",
      "Relationship                       640 non-null object\n",
      "RefSeq-Accn                        640 non-null object\n",
      "Assembly-Unit                      640 non-null object\n",
      "Sequence-Length                    640 non-null float64\n",
      "UCSC-style-name                    640 non-null object\n",
      "dtypes: float64(1), object(9)\n",
      "memory usage: 55.0+ KB\n"
     ]
    }
   ],
   "source": [
    "dna_data.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "640"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(dna_data)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "From the same https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.39/ page, under genome information, https://www.ncbi.nlm.nih.gov/genome/?term=txid9606[orgn], and in the Genome Assembly and Annotation report, https://www.ncbi.nlm.nih.gov/genome/genomes/51?, in the protein section at https://www.ncbi.nlm.nih.gov/genome/proteins/51?genome_assembly_id=582967, a comprehensive report of human proteins can be found. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "txt_file_protein = r\"ProteinTable51_582967.txt\"\n",
    "csv_file_protein = r\"ProteinTable51_582967.csv\"\n",
    "in_txt_protein = csv.reader(open(txt_file_protein, \"r\"), delimiter = '\\t')\n",
    "out_csv_protein = csv.writer(open(csv_file_protein, 'w'))\n",
    "out_csv_protein.writerows(in_txt_protein)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#Replicon Name</th>\n",
       "      <th>Replicon Accession</th>\n",
       "      <th>Start</th>\n",
       "      <th>Stop</th>\n",
       "      <th>Strand</th>\n",
       "      <th>GeneID</th>\n",
       "      <th>Locus</th>\n",
       "      <th>Protein product</th>\n",
       "      <th>Length</th>\n",
       "      <th>Protein name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>65080</td>\n",
       "      <td>10</td>\n",
       "      <td>NC_000010.11</td>\n",
       "      <td>74123810</td>\n",
       "      <td>74138379</td>\n",
       "      <td>-</td>\n",
       "      <td>26985</td>\n",
       "      <td>AP3M1</td>\n",
       "      <td>NP_036227.1</td>\n",
       "      <td>418</td>\n",
       "      <td>AP-3 complex subunit mu-1 isoform a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>95810</td>\n",
       "      <td>17</td>\n",
       "      <td>NC_000017.11</td>\n",
       "      <td>28703558</td>\n",
       "      <td>28704446</td>\n",
       "      <td>-</td>\n",
       "      <td>147011</td>\n",
       "      <td>PROCA1</td>\n",
       "      <td>NP_001291882.1</td>\n",
       "      <td>264</td>\n",
       "      <td>protein PROCA1 isoform e</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>117345</td>\n",
       "      <td>X</td>\n",
       "      <td>NC_000023.11</td>\n",
       "      <td>78656993</td>\n",
       "      <td>78658420</td>\n",
       "      <td>-</td>\n",
       "      <td>203430</td>\n",
       "      <td>RTL3</td>\n",
       "      <td>NP_689907.1</td>\n",
       "      <td>475</td>\n",
       "      <td>retrotransposon Gag-like protein 3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>25458</td>\n",
       "      <td>2</td>\n",
       "      <td>NC_000002.12</td>\n",
       "      <td>231710084</td>\n",
       "      <td>231712851</td>\n",
       "      <td>+</td>\n",
       "      <td>5757</td>\n",
       "      <td>PTMA</td>\n",
       "      <td>XP_011509821.1</td>\n",
       "      <td>262</td>\n",
       "      <td>prothymosin alpha isoform X2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>37396</td>\n",
       "      <td>4</td>\n",
       "      <td>NC_000004.12</td>\n",
       "      <td>168358147</td>\n",
       "      <td>168472699</td>\n",
       "      <td>-</td>\n",
       "      <td>91351</td>\n",
       "      <td>DDX60L</td>\n",
       "      <td>XP_005263402.2</td>\n",
       "      <td>1707</td>\n",
       "      <td>probable ATP-dependent RNA helicase DDX60-like...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3598</td>\n",
       "      <td>Un</td>\n",
       "      <td>NT_187670.1</td>\n",
       "      <td>66155</td>\n",
       "      <td>80026</td>\n",
       "      <td>-</td>\n",
       "      <td>3813</td>\n",
       "      <td>KIR3DS1</td>\n",
       "      <td>NP_001269100.1</td>\n",
       "      <td>275</td>\n",
       "      <td>killer cell immunoglobulin-like receptor 3DS1 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1553</td>\n",
       "      <td>Un</td>\n",
       "      <td>NT_167248.2</td>\n",
       "      <td>727236</td>\n",
       "      <td>728186</td>\n",
       "      <td>+</td>\n",
       "      <td>26716</td>\n",
       "      <td>OR2H1</td>\n",
       "      <td>NP_001304951.1</td>\n",
       "      <td>316</td>\n",
       "      <td>olfactory receptor 2H1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>24333</td>\n",
       "      <td>2</td>\n",
       "      <td>NC_000002.12</td>\n",
       "      <td>206307364</td>\n",
       "      <td>206311593</td>\n",
       "      <td>+</td>\n",
       "      <td>57683</td>\n",
       "      <td>ZDBF2</td>\n",
       "      <td>XP_016860065.1</td>\n",
       "      <td>1409</td>\n",
       "      <td>DBF4-type zinc finger-containing protein 2 iso...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>10173</td>\n",
       "      <td>1</td>\n",
       "      <td>NC_000001.11</td>\n",
       "      <td>47438136</td>\n",
       "      <td>47439623</td>\n",
       "      <td>+</td>\n",
       "      <td>2306</td>\n",
       "      <td>FOXD2</td>\n",
       "      <td>NP_004465.3</td>\n",
       "      <td>495</td>\n",
       "      <td>forkhead box protein D2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>59238</td>\n",
       "      <td>9</td>\n",
       "      <td>NC_000009.12</td>\n",
       "      <td>79573332</td>\n",
       "      <td>79725144</td>\n",
       "      <td>+</td>\n",
       "      <td>7091</td>\n",
       "      <td>TLE4</td>\n",
       "      <td>XP_016870556.1</td>\n",
       "      <td>829</td>\n",
       "      <td>transducin-like enhancer protein 4 isoform X11</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       #Replicon Name Replicon Accession      Start       Stop Strand  GeneID  \\\n",
       "65080              10       NC_000010.11   74123810   74138379      -   26985   \n",
       "95810              17       NC_000017.11   28703558   28704446      -  147011   \n",
       "117345              X       NC_000023.11   78656993   78658420      -  203430   \n",
       "25458               2       NC_000002.12  231710084  231712851      +    5757   \n",
       "37396               4       NC_000004.12  168358147  168472699      -   91351   \n",
       "3598               Un        NT_187670.1      66155      80026      -    3813   \n",
       "1553               Un        NT_167248.2     727236     728186      +   26716   \n",
       "24333               2       NC_000002.12  206307364  206311593      +   57683   \n",
       "10173               1       NC_000001.11   47438136   47439623      +    2306   \n",
       "59238               9       NC_000009.12   79573332   79725144      +    7091   \n",
       "\n",
       "          Locus Protein product  Length  \\\n",
       "65080     AP3M1     NP_036227.1     418   \n",
       "95810    PROCA1  NP_001291882.1     264   \n",
       "117345     RTL3     NP_689907.1     475   \n",
       "25458      PTMA  XP_011509821.1     262   \n",
       "37396    DDX60L  XP_005263402.2    1707   \n",
       "3598    KIR3DS1  NP_001269100.1     275   \n",
       "1553      OR2H1  NP_001304951.1     316   \n",
       "24333     ZDBF2  XP_016860065.1    1409   \n",
       "10173     FOXD2     NP_004465.3     495   \n",
       "59238      TLE4  XP_016870556.1     829   \n",
       "\n",
       "                                             Protein name  \n",
       "65080                 AP-3 complex subunit mu-1 isoform a  \n",
       "95810                            protein PROCA1 isoform e  \n",
       "117345                 retrotransposon Gag-like protein 3  \n",
       "25458                        prothymosin alpha isoform X2  \n",
       "37396   probable ATP-dependent RNA helicase DDX60-like...  \n",
       "3598    killer cell immunoglobulin-like receptor 3DS1 ...  \n",
       "1553                               olfactory receptor 2H1  \n",
       "24333   DBF4-type zinc finger-containing protein 2 iso...  \n",
       "10173                             forkhead box protein D2  \n",
       "59238      transducin-like enhancer protein 4 isoform X11  "
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "protein_data = pd.read_csv('ProteinTable51_582967.csv')\n",
    "protein_data.sample(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 119489 entries, 0 to 119488\n",
      "Data columns (total 10 columns):\n",
      "#Replicon Name        119489 non-null object\n",
      "Replicon Accession    119489 non-null object\n",
      "Start                 119489 non-null int64\n",
      "Stop                  119489 non-null int64\n",
      "Strand                119489 non-null object\n",
      "GeneID                119489 non-null int64\n",
      "Locus                 119489 non-null object\n",
      "Protein product       119489 non-null object\n",
      "Length                119489 non-null int64\n",
      "Protein name          119489 non-null object\n",
      "dtypes: int64(4), object(6)\n",
      "memory usage: 9.1+ MB\n"
     ]
    }
   ],
   "source": [
    "protein_data.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "119489"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(protein_data)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The protein and dna information is related through the RefSeq-Accn column on the dna_data table and the Replicon Accession column on the protein_data table. The two cells below show that many types of protein information is stored in chromosomes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th># Sequence-Name</th>\n",
       "      <th>Sequence-Role</th>\n",
       "      <th>Assigned-Molecule</th>\n",
       "      <th>Assigned-Molecule-Location/Type</th>\n",
       "      <th>GenBank-Accn</th>\n",
       "      <th>Relationship</th>\n",
       "      <th>RefSeq-Accn</th>\n",
       "      <th>Assembly-Unit</th>\n",
       "      <th>Sequence-Length</th>\n",
       "      <th>UCSC-style-name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      <td>12</td>\n",
       "      <td>assembled-molecule</td>\n",
       "      <td>12</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>CM000674.2</td>\n",
       "      <td>=</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>Primary Assembly</td>\n",
       "      <td>133275309.0</td>\n",
       "      <td>chr12</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   # Sequence-Name       Sequence-Role Assigned-Molecule  \\\n",
       "23              12  assembled-molecule                12   \n",
       "\n",
       "   Assigned-Molecule-Location/Type GenBank-Accn Relationship   RefSeq-Accn  \\\n",
       "23                      Chromosome   CM000674.2            =  NC_000012.12   \n",
       "\n",
       "       Assembly-Unit  Sequence-Length UCSC-style-name  \n",
       "23  Primary Assembly      133275309.0           chr12  "
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dna_data[dna_data['RefSeq-Accn'] == 'NC_000012.12']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#Replicon Name</th>\n",
       "      <th>Replicon Accession</th>\n",
       "      <th>Start</th>\n",
       "      <th>Stop</th>\n",
       "      <th>Strand</th>\n",
       "      <th>GeneID</th>\n",
       "      <th>Locus</th>\n",
       "      <th>Protein product</th>\n",
       "      <th>Length</th>\n",
       "      <th>Protein name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>74388</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>66883</td>\n",
       "      <td>175033</td>\n",
       "      <td>+</td>\n",
       "      <td>440073</td>\n",
       "      <td>IQSEC3</td>\n",
       "      <td>NP_001164209.1</td>\n",
       "      <td>1182</td>\n",
       "      <td>IQ motif and SEC7 domain-containing protein 3 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74389</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>66883</td>\n",
       "      <td>171308</td>\n",
       "      <td>+</td>\n",
       "      <td>440073</td>\n",
       "      <td>IQSEC3</td>\n",
       "      <td>XP_016874800.1</td>\n",
       "      <td>1041</td>\n",
       "      <td>IQ motif and SEC7 domain-containing protein 3 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74390</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>66883</td>\n",
       "      <td>171321</td>\n",
       "      <td>+</td>\n",
       "      <td>440073</td>\n",
       "      <td>IQSEC3</td>\n",
       "      <td>XP_011519262.1</td>\n",
       "      <td>1062</td>\n",
       "      <td>IQ motif and SEC7 domain-containing protein 3 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74391</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>66883</td>\n",
       "      <td>171308</td>\n",
       "      <td>+</td>\n",
       "      <td>440073</td>\n",
       "      <td>IQSEC3</td>\n",
       "      <td>XP_016874799.1</td>\n",
       "      <td>1041</td>\n",
       "      <td>IQ motif and SEC7 domain-containing protein 3 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74392</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>66883</td>\n",
       "      <td>174693</td>\n",
       "      <td>+</td>\n",
       "      <td>440073</td>\n",
       "      <td>IQSEC3</td>\n",
       "      <td>XP_011519260.1</td>\n",
       "      <td>1063</td>\n",
       "      <td>IQ motif and SEC7 domain-containing protein 3 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80340</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>133218885</td>\n",
       "      <td>133234356</td>\n",
       "      <td>-</td>\n",
       "      <td>647589</td>\n",
       "      <td>ANHX</td>\n",
       "      <td>XP_006719806.1</td>\n",
       "      <td>492</td>\n",
       "      <td>anomalous homeobox protein isoform X1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80341</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>133218885</td>\n",
       "      <td>133234356</td>\n",
       "      <td>-</td>\n",
       "      <td>647589</td>\n",
       "      <td>ANHX</td>\n",
       "      <td>XP_011533124.1</td>\n",
       "      <td>336</td>\n",
       "      <td>anomalous homeobox protein isoform X3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80342</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>133221223</td>\n",
       "      <td>133234356</td>\n",
       "      <td>-</td>\n",
       "      <td>647589</td>\n",
       "      <td>ANHX</td>\n",
       "      <td>XP_011533127.1</td>\n",
       "      <td>322</td>\n",
       "      <td>anomalous homeobox protein isoform X5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80343</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>133221223</td>\n",
       "      <td>133234356</td>\n",
       "      <td>-</td>\n",
       "      <td>647589</td>\n",
       "      <td>ANHX</td>\n",
       "      <td>XP_011533126.1</td>\n",
       "      <td>331</td>\n",
       "      <td>anomalous homeobox protein isoform X4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80344</td>\n",
       "      <td>12</td>\n",
       "      <td>NC_000012.12</td>\n",
       "      <td>133221223</td>\n",
       "      <td>133234356</td>\n",
       "      <td>-</td>\n",
       "      <td>647589</td>\n",
       "      <td>ANHX</td>\n",
       "      <td>XP_011533125.1</td>\n",
       "      <td>331</td>\n",
       "      <td>anomalous homeobox protein isoform X4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5957 rows × 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      #Replicon Name Replicon Accession      Start       Stop Strand  GeneID  \\\n",
       "74388             12       NC_000012.12      66883     175033      +  440073   \n",
       "74389             12       NC_000012.12      66883     171308      +  440073   \n",
       "74390             12       NC_000012.12      66883     171321      +  440073   \n",
       "74391             12       NC_000012.12      66883     171308      +  440073   \n",
       "74392             12       NC_000012.12      66883     174693      +  440073   \n",
       "...              ...                ...        ...        ...    ...     ...   \n",
       "80340             12       NC_000012.12  133218885  133234356      -  647589   \n",
       "80341             12       NC_000012.12  133218885  133234356      -  647589   \n",
       "80342             12       NC_000012.12  133221223  133234356      -  647589   \n",
       "80343             12       NC_000012.12  133221223  133234356      -  647589   \n",
       "80344             12       NC_000012.12  133221223  133234356      -  647589   \n",
       "\n",
       "        Locus Protein product  Length  \\\n",
       "74388  IQSEC3  NP_001164209.1    1182   \n",
       "74389  IQSEC3  XP_016874800.1    1041   \n",
       "74390  IQSEC3  XP_011519262.1    1062   \n",
       "74391  IQSEC3  XP_016874799.1    1041   \n",
       "74392  IQSEC3  XP_011519260.1    1063   \n",
       "...       ...             ...     ...   \n",
       "80340    ANHX  XP_006719806.1     492   \n",
       "80341    ANHX  XP_011533124.1     336   \n",
       "80342    ANHX  XP_011533127.1     322   \n",
       "80343    ANHX  XP_011533126.1     331   \n",
       "80344    ANHX  XP_011533125.1     331   \n",
       "\n",
       "                                            Protein name  \n",
       "74388  IQ motif and SEC7 domain-containing protein 3 ...  \n",
       "74389  IQ motif and SEC7 domain-containing protein 3 ...  \n",
       "74390  IQ motif and SEC7 domain-containing protein 3 ...  \n",
       "74391  IQ motif and SEC7 domain-containing protein 3 ...  \n",
       "74392  IQ motif and SEC7 domain-containing protein 3 ...  \n",
       "...                                                  ...  \n",
       "80340              anomalous homeobox protein isoform X1  \n",
       "80341              anomalous homeobox protein isoform X3  \n",
       "80342              anomalous homeobox protein isoform X5  \n",
       "80343              anomalous homeobox protein isoform X4  \n",
       "80344              anomalous homeobox protein isoform X4  \n",
       "\n",
       "[5957 rows x 10 columns]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "protein_data[protein_data['Replicon Accession']== 'NC_000012.12']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# Show plots as part of the notebook (this is a Jupyter-specific operation)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "# Standard library packages\n",
    "import os\n",
    "\n",
    "# Import Pandas and Seaborn\n",
    "import pandas as pd\n",
    "import seaborn as sns\n",
    "\n",
    "\n",
    "# Import Biopython tools for running local BLASTX\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast.Applications import NcbiblastpCommandline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Define paths to input and output directories\n",
    "db = os.path.join('blastdb')\n",
    "datadir = os.path.join(db, 'input')   # input\n",
    "outdir = os.path.join(db, 'output')  # output\n",
    "os.makedirs(outdir, exist_ok=True)  # create output directory if it doesn't exist"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'blastdb\\\\output'"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "outdir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Define paths to input and output files\n",
    "# query = os.path.join(datadir,name_of_file)# query sequence(s)\n",
    "                 # BLAST database \n",
    "# blastout = os.path.join(outdir, what_to_name_file)# BLAST output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "16"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.cpu_count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Help on class NcbiblastnCommandline in module Bio.Blast.Applications:\n",
      "\n",
      "class NcbiblastnCommandline(_NcbiblastMain2SeqCommandline)\n",
      " |  Wrapper for the NCBI BLAST+ program blastn (for nucleotides).\n",
      " |  \n",
      " |  With the release of BLAST+ (BLAST rewritten in C++ instead of C), the NCBI\n",
      " |  replaced the old blastall tool with separate tools for each of the searches.\n",
      " |  This wrapper therefore replaces BlastallCommandline with option -p blastn.\n",
      " |  \n",
      " |  For example, to run a search against the \"nt\" nucleotide database using the\n",
      " |  FASTA nucleotide file \"m_code.fasta\" as the query, with an expectation value\n",
      " |  cut off of 0.001, saving the output to a file in XML format:\n",
      " |  \n",
      " |  >>> from Bio.Blast.Applications import NcbiblastnCommandline\n",
      " |  >>> cline = NcbiblastnCommandline(query=\"m_cold.fasta\", db=\"nt\", strand=\"plus\",\n",
      " |  ...                               evalue=0.001, out=\"m_cold.xml\", outfmt=5)\n",
      " |  >>> cline\n",
      " |  NcbiblastnCommandline(cmd='blastn', out='m_cold.xml', outfmt=5, query='m_cold.fasta', db='nt', evalue=0.001, strand='plus')\n",
      " |  >>> print(cline)\n",
      " |  blastn -out m_cold.xml -outfmt 5 -query m_cold.fasta -db nt -evalue 0.001 -strand plus\n",
      " |  \n",
      " |  You would typically run the command line with cline() or via the Python\n",
      " |  subprocess module, as described in the Biopython tutorial.\n",
      " |  \n",
      " |  Method resolution order:\n",
      " |      NcbiblastnCommandline\n",
      " |      _NcbiblastMain2SeqCommandline\n",
      " |      _Ncbiblast2SeqCommandline\n",
      " |      _NcbiblastCommandline\n",
      " |      _NcbibaseblastCommandline\n",
      " |      Bio.Application.AbstractCommandline\n",
      " |      builtins.object\n",
      " |  \n",
      " |  Methods defined here:\n",
      " |  \n",
      " |  __init__(self, cmd='blastn', **kwargs)\n",
      " |      Initialize the class.\n",
      " |  \n",
      " |  ----------------------------------------------------------------------\n",
      " |  Methods inherited from Bio.Application.AbstractCommandline:\n",
      " |  \n",
      " |  __call__(self, stdin=None, stdout=True, stderr=True, cwd=None, env=None)\n",
      " |      Execute command, wait for it to finish, return (stdout, stderr).\n",
      " |      \n",
      " |      Runs the command line tool and waits for it to finish. If it returns\n",
      " |      a non-zero error level, an exception is raised. Otherwise two strings\n",
      " |      are returned containing stdout and stderr.\n",
      " |      \n",
      " |      The optional stdin argument should be a string of data which will be\n",
      " |      passed to the tool as standard input.\n",
      " |      \n",
      " |      The optional stdout and stderr argument may be filenames (string),\n",
      " |      but otherwise are treated as a booleans, and control if the output\n",
      " |      should be captured as strings (True, default), or ignored by sending\n",
      " |      it to /dev/null to avoid wasting memory (False). If sent to a file\n",
      " |      or ignored, then empty string(s) are returned.\n",
      " |      \n",
      " |      The optional cwd argument is a string giving the working directory\n",
      " |      to run the command from. See Python's subprocess module documentation\n",
      " |      for more details.\n",
      " |      \n",
      " |      The optional env argument is a dictionary setting the environment\n",
      " |      variables to be used in the new process. By default the current\n",
      " |      process' environment variables are used. See Python's subprocess\n",
      " |      module documentation for more details.\n",
      " |      \n",
      " |      Default example usage::\n",
      " |      \n",
      " |          from Bio.Emboss.Applications import WaterCommandline\n",
      " |          water_cmd = WaterCommandline(gapopen=10, gapextend=0.5,\n",
      " |                                       stdout=True, auto=True,\n",
      " |                                       asequence=\"a.fasta\", bsequence=\"b.fasta\")\n",
      " |          print(\"About to run: %s\" % water_cmd)\n",
      " |          std_output, err_output = water_cmd()\n",
      " |      \n",
      " |      This functionality is similar to subprocess.check_output() added in\n",
      " |      Python 2.7. In general if you require more control over running the\n",
      " |      command, use subprocess directly.\n",
      " |      \n",
      " |      As of Biopython 1.56, when the program called returns a non-zero error\n",
      " |      level, a custom ApplicationError exception is raised. This includes\n",
      " |      any stdout and stderr strings captured as attributes of the exception\n",
      " |      object, since they may be useful for diagnosing what went wrong.\n",
      " |  \n",
      " |  __repr__(self)\n",
      " |      Return a representation of the command line object for debugging.\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Emboss.Applications import WaterCommandline\n",
      " |      >>> cline = WaterCommandline(gapopen=10, gapextend=0.5)\n",
      " |      >>> cline.asequence = \"asis:ACCCGGGCGCGGT\"\n",
      " |      >>> cline.bsequence = \"asis:ACCCGAGCGCGGT\"\n",
      " |      >>> cline.outfile = \"temp_water.txt\"\n",
      " |      >>> print(cline)\n",
      " |      water -outfile=temp_water.txt -asequence=asis:ACCCGGGCGCGGT -bsequence=asis:ACCCGAGCGCGGT -gapopen=10 -gapextend=0.5\n",
      " |      >>> cline\n",
      " |      WaterCommandline(cmd='water', outfile='temp_water.txt', asequence='asis:ACCCGGGCGCGGT', bsequence='asis:ACCCGAGCGCGGT', gapopen=10, gapextend=0.5)\n",
      " |  \n",
      " |  __setattr__(self, name, value)\n",
      " |      Set attribute name to value (PRIVATE).\n",
      " |      \n",
      " |      This code implements a workaround for a user interface issue.\n",
      " |      Without this __setattr__ attribute-based assignment of parameters\n",
      " |      will silently accept invalid parameters, leading to known instances\n",
      " |      of the user assuming that parameters for the application are set,\n",
      " |      when they are not.\n",
      " |      \n",
      " |      >>> from Bio.Emboss.Applications import WaterCommandline\n",
      " |      >>> cline = WaterCommandline(gapopen=10, gapextend=0.5, stdout=True)\n",
      " |      >>> cline.asequence = \"a.fasta\"\n",
      " |      >>> cline.bsequence = \"b.fasta\"\n",
      " |      >>> cline.csequence = \"c.fasta\"\n",
      " |      Traceback (most recent call last):\n",
      " |      ...\n",
      " |      ValueError: Option name csequence was not found.\n",
      " |      >>> print(cline)\n",
      " |      water -stdout -asequence=a.fasta -bsequence=b.fasta -gapopen=10 -gapextend=0.5\n",
      " |      \n",
      " |      This workaround uses a whitelist of object attributes, and sets the\n",
      " |      object attribute list as normal, for these.  Other attributes are\n",
      " |      assumed to be parameters, and passed to the self.set_parameter method\n",
      " |      for validation and assignment.\n",
      " |  \n",
      " |  __str__(self)\n",
      " |      Make the commandline string with the currently set options.\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Emboss.Applications import WaterCommandline\n",
      " |      >>> cline = WaterCommandline(gapopen=10, gapextend=0.5)\n",
      " |      >>> cline.asequence = \"asis:ACCCGGGCGCGGT\"\n",
      " |      >>> cline.bsequence = \"asis:ACCCGAGCGCGGT\"\n",
      " |      >>> cline.outfile = \"temp_water.txt\"\n",
      " |      >>> print(cline)\n",
      " |      water -outfile=temp_water.txt -asequence=asis:ACCCGGGCGCGGT -bsequence=asis:ACCCGAGCGCGGT -gapopen=10 -gapextend=0.5\n",
      " |      >>> str(cline)\n",
      " |      'water -outfile=temp_water.txt -asequence=asis:ACCCGGGCGCGGT -bsequence=asis:ACCCGAGCGCGGT -gapopen=10 -gapextend=0.5'\n",
      " |  \n",
      " |  set_parameter(self, name, value=None)\n",
      " |      Set a commandline option for a program (OBSOLETE).\n",
      " |      \n",
      " |      Every parameter is available via a property and as a named\n",
      " |      keyword when creating the instance. Using either of these is\n",
      " |      preferred to this legacy set_parameter method which is now\n",
      " |      OBSOLETE, and likely to be DEPRECATED and later REMOVED in\n",
      " |      future releases.\n",
      " |  \n",
      " |  ----------------------------------------------------------------------\n",
      " |  Data descriptors inherited from Bio.Application.AbstractCommandline:\n",
      " |  \n",
      " |  __dict__\n",
      " |      dictionary for instance variables (if defined)\n",
      " |  \n",
      " |  __weakref__\n",
      " |      list of weak references to the object (if defined)\n",
      " |  \n",
      " |  ----------------------------------------------------------------------\n",
      " |  Data and other attributes inherited from Bio.Application.AbstractCommandline:\n",
      " |  \n",
      " |  parameters = None\n",
      "\n"
     ]
    }
   ],
   "source": [
    "help(NcbiblastnCommandline)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# packages in environment at C:\\Users\\eannefawcett\\Anaconda3\\envs\\learn-env:\n",
      "#\n",
      "# Name                    Version                   Build  Channel\n",
      "_py-xgboost-mutex         2.0                       cpu_0  \n",
      "asn1crypto                1.0.1                    py36_0  \n",
      "attrs                     19.3.0                     py_0  \n",
      "backcall                  0.1.0                    py36_0  \n",
      "beautifulsoup4            4.8.0                    py36_0  \n",
      "biopython                 1.76                     pypi_0    pypi\n",
      "blas                      1.0                         mkl  \n",
      "bleach                    3.1.0                    py36_0  \n",
      "boto                      2.49.0                   py36_0  \n",
      "boto3                     1.9.234                    py_0  \n",
      "botocore                  1.12.234                   py_0  \n",
      "bz2file                   0.98                     py36_1  \n",
      "ca-certificates           2019.8.28                     0  \n",
      "certifi                   2019.9.11                py36_0  \n",
      "cffi                      1.12.3           py36h7a1dbc1_0  \n",
      "chardet                   3.0.4                 py36_1003  \n",
      "colorama                  0.4.1                    py36_0  \n",
      "cryptography              2.7              py36h7a1dbc1_0  \n",
      "cryptography-vectors      2.7                        py_0  \n",
      "cycler                    0.10.0           py36h009560c_0  \n",
      "cython                    0.29.13          py36ha925a31_0  \n",
      "decorator                 4.4.0                    py36_1  \n",
      "defusedxml                0.6.0                      py_0  \n",
      "docutils                  0.15.2                   py36_0  \n",
      "entrypoints               0.3                      py36_0  \n",
      "freetype                  2.9.1                ha9979f8_1  \n",
      "gensim                    3.8.0            py36hf9181ef_0  \n",
      "h5py                      2.9.0            py36h5e291fa_0  \n",
      "hdf5                      1.10.4               h7ebc959_0  \n",
      "icc_rt                    2019.0.0             h0cc432a_1  \n",
      "icu                       58.2                 ha66f8fd_1  \n",
      "idna                      2.8                      py36_0  \n",
      "intel-openmp              2019.4                      245  \n",
      "ipykernel                 5.1.2            py36h39e3cac_0  \n",
      "ipython                   7.8.0            py36h39e3cac_0  \n",
      "ipython_genutils          0.2.0            py36h3c5d0ee_0  \n",
      "jedi                      0.15.1                   py36_0  \n",
      "jinja2                    2.10.3                     py_0  \n",
      "jmespath                  0.9.4                      py_0  \n",
      "joblib                    0.13.2                   py36_0  \n",
      "jpeg                      9b                   hb83a4c4_2  \n",
      "jsonschema                3.0.2                    py36_0  \n",
      "jupyter_client            5.3.3                    py36_1  \n",
      "jupyter_core              4.5.0                      py_0  \n",
      "kiwisolver                1.1.0            py36ha925a31_0  \n",
      "libpng                    1.6.37               h2a8f88b_0  \n",
      "libsodium                 1.0.16               h9d3ae62_0  \n",
      "libxgboost                0.90                          0  \n",
      "m2w64-gcc-libgfortran     5.3.0                         6  \n",
      "m2w64-gcc-libs            5.3.0                         7  \n",
      "m2w64-gcc-libs-core       5.3.0                         7  \n",
      "m2w64-gmp                 6.1.0                         2  \n",
      "m2w64-libwinpthread-git   5.0.0.4634.697f757               2  \n",
      "markupsafe                1.1.1            py36he774522_0  \n",
      "matplotlib                3.1.1            py36hc8f65d3_0  \n",
      "mistune                   0.8.4            py36he774522_0  \n",
      "mkl                       2019.4                      245  \n",
      "mkl-service               2.3.0            py36hb782905_0  \n",
      "mkl_fft                   1.0.14           py36h14836fe_0  \n",
      "mkl_random                1.1.0            py36h675688f_0  \n",
      "msys2-conda-epoch         20160418                      1  \n",
      "nb_conda                  2.2.1                    py36_0  \n",
      "nb_conda_kernels          2.2.2                    py36_0  \n",
      "nbconvert                 5.6.0                    py36_1  \n",
      "nbformat                  4.4.0            py36h3a5bc1b_0  \n",
      "networkx                  2.3                        py_0  \n",
      "nltk                      3.4.5                    py36_0  \n",
      "notebook                  6.0.1                    py36_0  \n",
      "numpy                     1.16.5           py36h19fb1c0_0  \n",
      "numpy-base                1.16.5           py36hc3f5095_0  \n",
      "openssl                   1.1.1d               he774522_2  \n",
      "pandas                    0.25.1           py36ha925a31_0  \n",
      "pandoc                    2.2.3.2                       0  \n",
      "pandocfilters             1.4.2                    py36_1  \n",
      "parso                     0.5.1                      py_0  \n",
      "patsy                     0.5.1                    py36_0  \n",
      "pexpect                   4.7.0                    py36_0  \n",
      "pickleshare               0.7.5                    py36_0  \n",
      "pip                       19.2.3                   py36_0  \n",
      "plotly                    4.1.1                      py_0  \n",
      "prometheus_client         0.7.1                      py_0  \n",
      "prompt_toolkit            2.0.10                     py_0  \n",
      "py-xgboost                0.90                     py36_0  \n",
      "py4j                      0.10.7                   py36_0  \n",
      "pycparser                 2.19                     py36_0  \n",
      "pydotplus                 2.0.2                    py36_1  \n",
      "pygments                  2.4.2                      py_0  \n",
      "pyopenssl                 19.0.0                   py36_0  \n",
      "pyparsing                 2.4.2                      py_0  \n",
      "pyqt                      5.9.2            py36h6538335_2  \n",
      "pyreadline                2.1                      py36_1  \n",
      "pyrsistent                0.15.4           py36he774522_0  \n",
      "pysocks                   1.7.1                    py36_0  \n",
      "pyspark                   2.4.4                      py_0  \n",
      "python                    3.6.9                h5500b2f_0  \n",
      "python-dateutil           2.8.0                    py36_0  \n",
      "pytz                      2019.3                     py_0  \n",
      "pywin32                   223              py36hfa6e2cd_1  \n",
      "pywinpty                  0.5.5                 py36_1000  \n",
      "pyzmq                     18.1.0           py36ha925a31_0  \n",
      "qt                        5.9.7            vc14h73c81de_0  \n",
      "requests                  2.22.0                   py36_0  \n",
      "retrying                  1.3.3                    py36_2  \n",
      "s3transfer                0.2.1                    py36_0  \n",
      "scikit-learn              0.21.3           py36h6288b17_0  \n",
      "scipy                     1.3.1            py36h29ff71c_0  \n",
      "seaborn                   0.9.0                    py36_0  \n",
      "send2trash                1.5.0                    py36_0  \n",
      "setuptools                41.4.0                   py36_0  \n",
      "simplegeneric             0.8.1                    py36_2  \n",
      "sip                       4.19.8           py36h6538335_0  \n",
      "six                       1.12.0                   py36_0  \n",
      "smart_open                1.8.4                      py_0  \n",
      "soupsieve                 1.9.3                    py36_0  \n",
      "sqlite                    3.30.0               he774522_0  \n",
      "statsmodels               0.10.1           py36h8c2d366_0  \n",
      "terminado                 0.8.2                    py36_0  \n",
      "testpath                  0.4.2                    py36_0  \n",
      "tk                        8.6.8                hfa6e2cd_0  \n",
      "tornado                   6.0.3            py36he774522_0  \n",
      "tqdm                      4.36.1                     py_0  \n",
      "traitlets                 4.3.3                    py36_0  \n",
      "urllib3                   1.24.2                   py36_0  \n",
      "vc                        14.1                 h0510ff6_4  \n",
      "vs2015_runtime            14.16.27012          hf0eaf9b_0  \n",
      "wcwidth                   0.1.7            py36h3d5aa90_0  \n",
      "webencodings              0.5.1                    py36_1  \n",
      "wheel                     0.33.6                   py36_0  \n",
      "win_inet_pton             1.1.0                    py36_0  \n",
      "wincertstore              0.2              py36h7fe50ca_0  \n",
      "winpty                    0.4.3                         4  \n",
      "xlrd                      1.2.0                    py36_0  \n",
      "xz                        5.2.4                h2fa13f4_4  \n",
      "zeromq                    4.3.1                h33f27b4_3  \n",
      "zlib                      1.2.11               h62dcd97_3  \n",
      "\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "conda list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "import Bio"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "ename": "ApplicationError",
     "evalue": "Non-zero return code 3 from 'blastn -out blastdb\\\\output\\\\output_NC_000012.12.txt -outfmt 6 -query blastdb\\\\input\\\\NC_000012.12.txt -db nt -num_threads 4 -task blastn -window_masker_taxid 9606', message 'BLAST engine error: Warning: NCBI C++ Exception:'",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mApplicationError\u001b[0m                          Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-39-a1adcb6c3fb0>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      4\u001b[0m \u001b[0mblastout\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mos\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0moutdir\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'output_NC_000012.12.txt'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      5\u001b[0m \u001b[0mcline\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mNcbiblastnCommandline\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mquery\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mquery\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mout\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mblastout\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0moutfmt\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m6\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mdb\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;34m'nt'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mtask\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;34m'blastn'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mnum_threads\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m4\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mwindow_masker_taxid\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m9606\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 6\u001b[1;33m \u001b[0mstdout\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mstderr\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mcline\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      7\u001b[0m \u001b[1;31m# Check STDOUT, STDERR\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      8\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"STDOUT: %s\"\u001b[0m \u001b[1;33m%\u001b[0m \u001b[0mstdout\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\envs\\learn-env\\lib\\site-packages\\Bio\\Application\\__init__.py\u001b[0m in \u001b[0;36m__call__\u001b[1;34m(self, stdin, stdout, stderr, cwd, env)\u001b[0m\n\u001b[0;32m    529\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0mreturn_code\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    530\u001b[0m             raise ApplicationError(return_code, str(self),\n\u001b[1;32m--> 531\u001b[1;33m                                    stdout_str, stderr_str)\n\u001b[0m\u001b[0;32m    532\u001b[0m         \u001b[1;32mreturn\u001b[0m \u001b[0mstdout_str\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mstderr_str\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    533\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mApplicationError\u001b[0m: Non-zero return code 3 from 'blastn -out blastdb\\\\output\\\\output_NC_000012.12.txt -outfmt 6 -query blastdb\\\\input\\\\NC_000012.12.txt -db nt -num_threads 4 -task blastn -window_masker_taxid 9606', message 'BLAST engine error: Warning: NCBI C++ Exception:'"
     ]
    }
   ],
   "source": [
    "import time\n",
    "start = time.time()\n",
    "query = os.path.join(datadir,'NC_000012.12.txt')\n",
    "blastout = os.path.join(outdir, 'output_NC_000012.12.txt')\n",
    "cline = NcbiblastnCommandline(query=query, out=blastout, outfmt=6, db='nt', task='blastn', num_threads=4, window_masker_taxid=9606)\n",
    "stdout, stderr = cline()\n",
    "# Check STDOUT, STDERR\n",
    "print(\"STDOUT: %s\" % stdout)\n",
    "print(\"STDERR: %s\" % stderr)\n",
    "print('Took', time.time()-start, 'seconds.')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'blastdb\\\\input\\\\NC_000012.12.txt'"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.path.exists(query)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Help on class NcbiblastpCommandline in module Bio.Blast.Applications:\n",
      "\n",
      "class NcbiblastpCommandline(_NcbiblastMain2SeqCommandline)\n",
      " |  Create a commandline for the NCBI BLAST+ program blastp (for proteins).\n",
      " |  \n",
      " |  With the release of BLAST+ (BLAST rewritten in C++ instead of C), the NCBI\n",
      " |  replaced the old blastall tool with separate tools for each of the searches.\n",
      " |  This wrapper therefore replaces BlastallCommandline with option -p blastp.\n",
      " |  \n",
      " |  >>> from Bio.Blast.Applications import NcbiblastpCommandline\n",
      " |  >>> cline = NcbiblastpCommandline(query=\"rosemary.pro\", db=\"nr\",\n",
      " |  ...                               evalue=0.001, remote=True, ungapped=True)\n",
      " |  >>> cline\n",
      " |  NcbiblastpCommandline(cmd='blastp', query='rosemary.pro', db='nr', evalue=0.001, remote=True, ungapped=True)\n",
      " |  >>> print(cline)\n",
      " |  blastp -query rosemary.pro -db nr -evalue 0.001 -remote -ungapped\n",
      " |  \n",
      " |  You would typically run the command line with cline() or via the Python\n",
      " |  subprocess module, as described in the Biopython tutorial.\n",
      " |  \n",
      " |  Method resolution order:\n",
      " |      NcbiblastpCommandline\n",
      " |      _NcbiblastMain2SeqCommandline\n",
      " |      _Ncbiblast2SeqCommandline\n",
      " |      _NcbiblastCommandline\n",
      " |      _NcbibaseblastCommandline\n",
      " |      Bio.Application.AbstractCommandline\n",
      " |      builtins.object\n",
      " |  \n",
      " |  Methods defined here:\n",
      " |  \n",
      " |  __init__(self, cmd='blastp', **kwargs)\n",
      " |      Initialize the class.\n",
      " |  \n",
      " |  ----------------------------------------------------------------------\n",
      " |  Methods inherited from Bio.Application.AbstractCommandline:\n",
      " |  \n",
      " |  __call__(self, stdin=None, stdout=True, stderr=True, cwd=None, env=None)\n",
      " |      Execute command, wait for it to finish, return (stdout, stderr).\n",
      " |      \n",
      " |      Runs the command line tool and waits for it to finish. If it returns\n",
      " |      a non-zero error level, an exception is raised. Otherwise two strings\n",
      " |      are returned containing stdout and stderr.\n",
      " |      \n",
      " |      The optional stdin argument should be a string of data which will be\n",
      " |      passed to the tool as standard input.\n",
      " |      \n",
      " |      The optional stdout and stderr argument may be filenames (string),\n",
      " |      but otherwise are treated as a booleans, and control if the output\n",
      " |      should be captured as strings (True, default), or ignored by sending\n",
      " |      it to /dev/null to avoid wasting memory (False). If sent to a file\n",
      " |      or ignored, then empty string(s) are returned.\n",
      " |      \n",
      " |      The optional cwd argument is a string giving the working directory\n",
      " |      to run the command from. See Python's subprocess module documentation\n",
      " |      for more details.\n",
      " |      \n",
      " |      The optional env argument is a dictionary setting the environment\n",
      " |      variables to be used in the new process. By default the current\n",
      " |      process' environment variables are used. See Python's subprocess\n",
      " |      module documentation for more details.\n",
      " |      \n",
      " |      Default example usage::\n",
      " |      \n",
      " |          from Bio.Emboss.Applications import WaterCommandline\n",
      " |          water_cmd = WaterCommandline(gapopen=10, gapextend=0.5,\n",
      " |                                       stdout=True, auto=True,\n",
      " |                                       asequence=\"a.fasta\", bsequence=\"b.fasta\")\n",
      " |          print(\"About to run: %s\" % water_cmd)\n",
      " |          std_output, err_output = water_cmd()\n",
      " |      \n",
      " |      This functionality is similar to subprocess.check_output() added in\n",
      " |      Python 2.7. In general if you require more control over running the\n",
      " |      command, use subprocess directly.\n",
      " |      \n",
      " |      As of Biopython 1.56, when the program called returns a non-zero error\n",
      " |      level, a custom ApplicationError exception is raised. This includes\n",
      " |      any stdout and stderr strings captured as attributes of the exception\n",
      " |      object, since they may be useful for diagnosing what went wrong.\n",
      " |  \n",
      " |  __repr__(self)\n",
      " |      Return a representation of the command line object for debugging.\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Emboss.Applications import WaterCommandline\n",
      " |      >>> cline = WaterCommandline(gapopen=10, gapextend=0.5)\n",
      " |      >>> cline.asequence = \"asis:ACCCGGGCGCGGT\"\n",
      " |      >>> cline.bsequence = \"asis:ACCCGAGCGCGGT\"\n",
      " |      >>> cline.outfile = \"temp_water.txt\"\n",
      " |      >>> print(cline)\n",
      " |      water -outfile=temp_water.txt -asequence=asis:ACCCGGGCGCGGT -bsequence=asis:ACCCGAGCGCGGT -gapopen=10 -gapextend=0.5\n",
      " |      >>> cline\n",
      " |      WaterCommandline(cmd='water', outfile='temp_water.txt', asequence='asis:ACCCGGGCGCGGT', bsequence='asis:ACCCGAGCGCGGT', gapopen=10, gapextend=0.5)\n",
      " |  \n",
      " |  __setattr__(self, name, value)\n",
      " |      Set attribute name to value (PRIVATE).\n",
      " |      \n",
      " |      This code implements a workaround for a user interface issue.\n",
      " |      Without this __setattr__ attribute-based assignment of parameters\n",
      " |      will silently accept invalid parameters, leading to known instances\n",
      " |      of the user assuming that parameters for the application are set,\n",
      " |      when they are not.\n",
      " |      \n",
      " |      >>> from Bio.Emboss.Applications import WaterCommandline\n",
      " |      >>> cline = WaterCommandline(gapopen=10, gapextend=0.5, stdout=True)\n",
      " |      >>> cline.asequence = \"a.fasta\"\n",
      " |      >>> cline.bsequence = \"b.fasta\"\n",
      " |      >>> cline.csequence = \"c.fasta\"\n",
      " |      Traceback (most recent call last):\n",
      " |      ...\n",
      " |      ValueError: Option name csequence was not found.\n",
      " |      >>> print(cline)\n",
      " |      water -stdout -asequence=a.fasta -bsequence=b.fasta -gapopen=10 -gapextend=0.5\n",
      " |      \n",
      " |      This workaround uses a whitelist of object attributes, and sets the\n",
      " |      object attribute list as normal, for these.  Other attributes are\n",
      " |      assumed to be parameters, and passed to the self.set_parameter method\n",
      " |      for validation and assignment.\n",
      " |  \n",
      " |  __str__(self)\n",
      " |      Make the commandline string with the currently set options.\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Emboss.Applications import WaterCommandline\n",
      " |      >>> cline = WaterCommandline(gapopen=10, gapextend=0.5)\n",
      " |      >>> cline.asequence = \"asis:ACCCGGGCGCGGT\"\n",
      " |      >>> cline.bsequence = \"asis:ACCCGAGCGCGGT\"\n",
      " |      >>> cline.outfile = \"temp_water.txt\"\n",
      " |      >>> print(cline)\n",
      " |      water -outfile=temp_water.txt -asequence=asis:ACCCGGGCGCGGT -bsequence=asis:ACCCGAGCGCGGT -gapopen=10 -gapextend=0.5\n",
      " |      >>> str(cline)\n",
      " |      'water -outfile=temp_water.txt -asequence=asis:ACCCGGGCGCGGT -bsequence=asis:ACCCGAGCGCGGT -gapopen=10 -gapextend=0.5'\n",
      " |  \n",
      " |  set_parameter(self, name, value=None)\n",
      " |      Set a commandline option for a program (OBSOLETE).\n",
      " |      \n",
      " |      Every parameter is available via a property and as a named\n",
      " |      keyword when creating the instance. Using either of these is\n",
      " |      preferred to this legacy set_parameter method which is now\n",
      " |      OBSOLETE, and likely to be DEPRECATED and later REMOVED in\n",
      " |      future releases.\n",
      " |  \n",
      " |  ----------------------------------------------------------------------\n",
      " |  Data descriptors inherited from Bio.Application.AbstractCommandline:\n",
      " |  \n",
      " |  __dict__\n",
      " |      dictionary for instance variables (if defined)\n",
      " |  \n",
      " |  __weakref__\n",
      " |      list of weak references to the object (if defined)\n",
      " |  \n",
      " |  ----------------------------------------------------------------------\n",
      " |  Data and other attributes inherited from Bio.Application.AbstractCommandline:\n",
      " |  \n",
      " |  parameters = None\n",
      "\n"
     ]
    }
   ],
   "source": [
    "help(NcbiblastpCommandline)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import time\n",
    "start = time.time()\n",
    "query = os.path.join(datadir,'NP_001164209.1.txt')\n",
    "blastout = os.path.join(outdir, 'output_NP_001164209.1.txt')\n",
    "cline = NcbiblastpCommandline(query=query, out=blastout, outfmt=6, db='refseq_protein', task='blastp', num_threads=4)\n",
    "stdout, stderr = cline()\n",
    "# Check STDOUT, STDERR\n",
    "print(\"STDOUT: %s\" % stdout)\n",
    "print(\"STDERR: %s\" % stderr)\n",
    "print('Took', time.time()-start, 'seconds.')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "query"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\"C:\\Users\\eannefawcett\\Desktop\\Flatiron\\Module 5\\project\\blastdb\\input\\NP_001164209.1.txt\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.path.exists(query)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "References\n",
    "Stephen F. Altschul, Thomas L. Madden, Alejandro A. Schäffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), \"Gapped BLAST and PSI-BLAST: a new generation of protein database search programs\", Nucleic Acids Res. 25:3389-3402.\n",
    "https://towardsdatascience.com/processing-xml-in-python-elementtree-c8992941efd2 # processing xml data\n",
    "https://stackoverflow.com/questions/10220412/convert-tab-delimited-txt-file-into-a-csv-file-using-python # import of a text file\n",
    "https://www.ncbi.nlm.nih.gov/assembly/model/\n",
    "ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/405/GCF_000001405.39_GRCh38.p13/GCF_000001405.39_GRCh38.p13_assembly_report.txt # dna data\n",
    "https://www.ncbi.nlm.nih.gov/genomes/Genome2BE/genome2srv.cgi?action=GetFeatures4Grid&amp;download=on&amp;type=Proteins&amp;genome_id=51&genome_assembly_id=582967&mode=2&is_locus=1&is_locus_tag=0&optcols=1,0,0,0,0 # protein data\n",
    "https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&PAGE_TYPE=BlastDocs&DOC_TYPE=References\n",
    "https://www.ncbi.nlm.nih.gov/pubmed/20003500?dopt=Citation # paper for the creation of blast+\n",
    "https://www.ncbi.nlm.nih.gov/books/NBK52637/ # stand alone blast+ set up for pc\n",
    "ftp://ftp.ncbi.nlm.nih.gov/blast/db/ # data base location\n",
    "http://seqanswers.com/forums/showthread.php?t=44370 # troubleshooting database set up\n",
    "https://widdowquinn.github.io/2018-03-06-ibioic/02-sequence_databases/03-programming_for_blast.html # python/blast+ integration"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "learn-env",
   "language": "python",
   "name": "learn-env"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
